{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "from Bio import Entrez\n",
    "import xml.etree.ElementTree as ET\n",
    "\n",
    "handle = Entrez.efetch(db=\"pmc\", id=5742488, retmode='xml')\n",
    "record = handle.read()\n",
    "handle.close()\n",
    "root = ET.fromstring(record)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TITLE:  Host Specificity for Bacterial, Archaeal and Fungal Communities Determined for High- and Low-Microbial Abundance Sponge Species in Two Genera\nABSTRACT:  Sponges are engaged in intimate symbioses with a diversity of microorganisms from all three domains of life, namely Bacteria, Archaea and Eukarya. Sponges have been well studied and categorized for their bacterial communities, some displaying a high microbial abundance (HMA), while others show low microbial abundance (LMA). However, the associated Archaea and Eukarya have remained relatively understudied. We assessed the bacterial, archaeal and eukaryotic diversities in the LMA sponge species \n<?xml version=\"1.0\" ?>\n<!DOCTYPE pmc-articleset PUBLIC \"-//NLM//DTD ARTICLE SET 2.0//EN\" \"https://dtd.nlm.nih.gov/ncbi/pmc/articleset/nlm-articleset-2.0.dtd\">\n<pmc-articleset><article xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" article-type=\"research-article\">\n  <?properties open_access?>\n  <front>\n    <journal-meta>\n      <journal-id journal-id-type=\"nlm-ta\">Front Microbiol</journal-id>\n      <journal-id journal-id-type=\"iso-abbrev\">Front Microbiol</journal-id>\n      <journal-id journal-id-type=\"publisher-id\">Front. Microbiol.</journal-id>\n      <journal-title-group>\n        <journal-title>Frontiers in Microbiology</journal-title>\n      </journal-title-group>\n      <issn pub-type=\"epub\">1664-302X</issn>\n      <publisher>\n        <publisher-name>Frontiers Media S.A.</publisher-name>\n      </publisher>\n    </journal-meta>\n    <article-meta>\n      <article-id pub-id-type=\"pmid\">29326681</article-id>\n      <article-id pub-id-type=\"pmc\">5742488</article-id>\n      <article-id pub-id-type=\"doi\">10.3389/fmicb.2017.02560</article-id>\n      <article-categories>\n        <subj-group subj-group-type=\"heading\">\n          <subject>Microbiology</subject>\n          <subj-group>\n            <subject>Original Research</subject>\n          </subj-group>\n        </subj-group>\n      </article-categories>\n      <title-group>\n        <article-title>Host Specificity for Bacterial, Archaeal and Fungal Communities Determined for High- and Low-Microbial Abundance Sponge Species in Two Genera</article-title>\n      </title-group>\n      <contrib-group>\n        <contrib contrib-type=\"author\">\n          <name>\n            <surname>Chaib De Mares</surname>\n            <given-names>Maryam</given-names>\n          </name>\n          <xref ref-type=\"aff\" rid=\"aff1\">\n            <sup>1</sup>\n          </xref>\n          <xref ref-type=\"author-notes\" rid=\"fn001\">\n            <sup>*</sup>\n          </xref>\n          <uri xlink:type=\"simple\" xlink:href=\"http://loop.frontiersin.org/people/476002/overview\"/>\n        </contrib>\n        <contrib contrib-type=\"author\">\n          <name>\n            <surname>Sipkema</surname>\n            <given-names>Detmer</given-names>\n          </name>\n          <xref ref-type=\"aff\" rid=\"aff2\">\n            <sup>2</sup>\n          </xref>\n          <uri xlink:type=\"simple\" xlink:href=\"http://loop.frontiersin.org/people/116529/overview\"/>\n        </contrib>\n        <contrib contrib-type=\"author\">\n          <name>\n            <surname>Huang</surname>\n            <given-names>Sixing</given-names>\n          </name>\n          <xref ref-type=\"aff\" rid=\"aff3\">\n            <sup>3</sup>\n          </xref>\n        </contrib>\n        <contrib contrib-type=\"author\">\n          <name>\n            <surname>Bunk</surname>\n            <given-names>Boyke</given-names>\n          </name>\n          <xref ref-type=\"aff\" rid=\"aff3\">\n            <sup>3</sup>\n          </xref>\n          <xref ref-type=\"aff\" rid=\"aff4\">\n            <sup>4</sup>\n          </xref>\n        </contrib>\n        <contrib contrib-type=\"author\">\n          <name>\n            <surname>Overmann</surname>\n            <given-names>J&#xF6;rg</given-names>\n          </name>\n          <xref ref-type=\"aff\" rid=\"aff3\">\n            <sup>3</sup>\n          </xref>\n          <xref ref-type=\"aff\" rid=\"aff4\">\n            <sup>4</sup>\n          </xref>\n          <uri xlink:type=\"simple\" xlink:href=\"http://loop.frontiersin.org/people/28868/overview\"/>\n        </contrib>\n        <contrib contrib-type=\"author\">\n          <name>\n            <surname>van Elsas</surname>\n            <given-names>Jan Dirk</given-names>\n          </name>\n          <xref ref-type=\"aff\" rid=\"aff1\">\n            <sup>1</sup>\n          </xref>\n          <uri xlink:type=\"simple\" xlink:href=\"http://loop.frontiersin.org/people/188611/overview\"/>\n        </contrib>\n      </contrib-group>\n      <aff id=\"aff1\"><sup>1</sup><institution>Microbial Ecology Cluster, Groningen Institute for Evolutionary Life Sciences, University of Groningen</institution>, <addr-line>Groningen</addr-line>, <country>Netherlands</country></aff>\n      <aff id=\"aff2\"><sup>2</sup><institution>Laboratory of Microbiology, Wageningen University</institution>, <addr-line>Wageningen</addr-line>, <country>Netherlands</country></aff>\n      <aff id=\"aff3\"><sup>3</sup><institution>Leibniz-Institut Deutsche Sammlung von Mikroorganismen und Zellkulturen</institution>, <addr-line>Braunschweig</addr-line>, <country>Germany</country></aff>\n      <aff id=\"aff4\"><sup>4</sup><institution>German Centre of Infection Research (DZIF), Partner site Hannover-Braunschweig</institution>, <addr-line>Braunschweig</addr-line>, <country>Germany</country></aff>\n      <author-notes>\n        <fn fn-type=\"edited-by\">\n          <p>Edited by: Robert Brucker, Rowland Institute at Harvard, United States</p>\n        </fn>\n        <fn fn-type=\"edited-by\">\n          <p>Reviewed by: Robert W. Thacker, Stony Brook University, United States; Torsten Thomas, University of New South Wales, Australia</p>\n        </fn>\n        <corresp id=\"fn001\">*Correspondence: Maryam Chaib De Mares <email xlink:type=\"simple\">m.chaib.de.mares@rug.nl</email></corresp>\n        <fn fn-type=\"other\" id=\"fn002\">\n          <p>This article was submitted to Microbial Symbioses, a section of the journal Frontiers in Microbiology</p>\n        </fn>\n      </author-notes>\n      <pub-date pub-type=\"epub\">\n        <day>20</day>\n        <month>12</month>\n        <year>2017</year>\n      </pub-date>\n      <pub-date pub-type=\"collection\">\n        <year>2017</year>\n      </pub-date>\n      <volume>8</volume>\n      <elocation-id>2560</elocation-id>\n      <history>\n        <date date-type=\"received\">\n          <day>12</day>\n          <month>9</month>\n          <year>2017</year>\n        </date>\n        <date date-type=\"accepted\">\n          <day>11</day>\n          <month>12</month>\n          <year>2017</year>\n        </date>\n      </history>\n      <permissions>\n        <copyright-statement>Copyright &#xA9; 2017 Chaib De Mares, Sipkema, Huang, Bunk, Overmann and van Elsas.</copyright-statement>\n        <copyright-year>2017</copyright-year>\n        <copyright-holder>Chaib De Mares, Sipkema, Huang, Bunk, Overmann and van Elsas</copyright-holder>\n        <license xlink:href=\"http://creativecommons.org/licenses/by/4.0/\">\n          <license-p>This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) or licensor are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</license-p>\n        </license>\n      </permissions>\n      <abstract>\n        <p>Sponges are engaged in intimate symbioses with a diversity of microorganisms from all three domains of life, namely Bacteria, Archaea and Eukarya. Sponges have been well studied and categorized for their bacterial communities, some displaying a high microbial abundance (HMA), while others show low microbial abundance (LMA). However, the associated Archaea and Eukarya have remained relatively understudied. We assessed the bacterial, archaeal and eukaryotic diversities in the LMA sponge species <italic>Dysidea avara</italic> and <italic>Dysidea etheria</italic> by deep amplicon sequencing, and compared the results to those in the HMA sponges <italic>Aplysina aerophoba</italic> and <italic>Aplysina cauliformis</italic>. <italic>D. avara</italic> and <italic>A. aerophoba</italic> are sympatric in the Mediterranean Sea, while <italic>D. etheria</italic> and <italic>A. cauliformis</italic> are sympatric in the Caribbean Sea. The bacterial communities followed a host-specific pattern, with host species identity explaining most of the variation among samples. We identified OTUs shared by the <italic>Aplysina</italic> species that support a more ancient association of these microbes, before the split of the two species studied here. These shared OTUs are suitable targets for future studies of the microbial traits that mediate interactions with their hosts. Even though the archaeal communities were not as rich as the bacterial ones, we found a remarkable diversification and specificity of OTUs of the family Cenarchaeaceae and the genus <italic>Nitrosopumilus</italic> in all four sponge species studied. Similarly, the differences in fungal communities were driven by sponge identity. The structures of the communities of small eukaryotes such as dinophytes and ciliophores (alveolates), and stramenopiles, could not be explained by either sponge host, sponge genus or geographic location. Our analyses suggest that the host specificity that was previously described for sponge bacterial communities also extends to the archaeal and fungal communities, but not to other microbial eukaryotes.</p>\n      </abstract>\n      <kwd-group>\n        <kwd>\n          <italic>Aplysina</italic>\n        </kwd>\n        <kwd>\n          <italic>Dysidea</italic>\n        </kwd>\n        <kwd>microbial diversity</kwd>\n        <kwd>three-domain microbial communities</kwd>\n        <kwd>sponges</kwd>\n      </kwd-group>\n      <funding-group>\n        <award-group>\n          <funding-source id=\"cn001\">Seventh Framework Programme<named-content content-type=\"fundref-id\">10.13039/100011102</named-content></funding-source>\n          <award-id rid=\"cn001\">607786</award-id>\n        </award-group>\n      </funding-group>\n      <counts>\n        <fig-count count=\"3\"/>\n        <table-count count=\"1\"/>\n        <equation-count count=\"0\"/>\n        <ref-count count=\"67\"/>\n        <page-count count=\"13\"/>\n        <word-count count=\"10400\"/>\n      </counts>\n    </article-meta>\n  </front>\n  <body>\n    <sec sec-type=\"intro\" id=\"s1\">\n      <title>Introduction</title>\n      <p>Sponges (phylum Porifera) feature an intimate symbiosis with diverse microorganisms (Taylor et al., <xref rid=\"B54\" ref-type=\"bibr\">2007</xref>; Simister et al., <xref rid=\"B49\" ref-type=\"bibr\">2012</xref>). Although some species occur in freshwater, we here place a focus on marine sponges. These are sessile filter feeders that acquire organic matter and microorganisms from seawater by pumping large volumes of water through their aquiferous system (Vogel, <xref rid=\"B62\" ref-type=\"bibr\">1977</xref>). In the process, the acquired microorganisms become transferred into the mesohyl tissue, where they are eventually ingested by archaeocytes (Simpson, <xref rid=\"B50\" ref-type=\"bibr\">1984</xref>). Microorganisms can, however, also survive in the mesohyl tissue and become established as part of the sponge-associated microbiota. This process of translocation of microorganisms from seawater to the sponge is referred to as horizontal transmission. Alternatively, sponges may acquire their associated microbiota by means of vertical transmission, through incorporation of microbial cells in egg cells or other reproductive stages. Vertical transmission in sponges has been observed for Bacteria (Ereskovsky et al., <xref rid=\"B19\" ref-type=\"bibr\">2005</xref>; Enticknap et al., <xref rid=\"B18\" ref-type=\"bibr\">2006</xref>; De Caralt et al., <xref rid=\"B13\" ref-type=\"bibr\">2007</xref>; Lee et al., <xref rid=\"B34\" ref-type=\"bibr\">2009</xref>), Archaea (Sharp et al., <xref rid=\"B48\" ref-type=\"bibr\">2007</xref>; Steger et al., <xref rid=\"B53\" ref-type=\"bibr\">2008</xref>) and even yeasts (Maldonado et al., <xref rid=\"B38\" ref-type=\"bibr\">2005</xref>). Consequently, the acquisition of similar sponge-associated microbes can occur both vertically and horizontally (Sipkema et al., <xref rid=\"B52\" ref-type=\"bibr\">2015</xref>).</p>\n      <p>Microorganisms enclosed within the mesohyl are physically separated from the surrounding seawater by the sponge pinacoderm. Sponges have been documented to harbor a complex community comprising microbial representatives from all three domains of life, the Bacteria, Archaea, and Eukarya, and can contain viruses as well. To date, thousands of bacterial and archaeal species have been identified within this association, being spread over more than 52 phyla (Schmitt et al., <xref rid=\"B46\" ref-type=\"bibr\">2011</xref>; Webster and Taylor, <xref rid=\"B64\" ref-type=\"bibr\">2012</xref>; Thomas et al., <xref rid=\"B56\" ref-type=\"bibr\">2016</xref>). In addition, several microbial eukaryotes including Fungi (Lee et al., <xref rid=\"B35\" ref-type=\"bibr\">2001</xref>; Usher et al., <xref rid=\"B59\" ref-type=\"bibr\">2007</xref>; Usher, <xref rid=\"B58\" ref-type=\"bibr\">2008</xref>; Webster and Blackall, <xref rid=\"B63\" ref-type=\"bibr\">2009</xref>; Simister et al., <xref rid=\"B49\" ref-type=\"bibr\">2012</xref>; Rodr&#xED;guez-Marconi et al., <xref rid=\"B45\" ref-type=\"bibr\">2015</xref>) have been reported.</p>\n      <p>Sponges differ in the numbers of bacteria they harbor. Sponges with dense tissues, and poorly irrigated ones, contain larger numbers of bacteria than well-irrigated ones. These different sponge types are referred to as high microbial abundance (HMA) or &#x201C;bacteriosponges,&#x201D; and low microbial abundance (LMA) sponges, respectively (Hentschel et al., <xref rid=\"B27\" ref-type=\"bibr\">2006</xref>). HMA sponges have densities between 10<sup>8</sup> and 10<sup>10</sup> of bacterial cells per gram of wet sponge weight, resulting in many cases in up to 40% of the sponge biomass being composed of microorganisms (Vacelet and Donadey, <xref rid=\"B61\" ref-type=\"bibr\">1977</xref>; Usher et al., <xref rid=\"B60\" ref-type=\"bibr\">2004</xref>), while LMA sponges have densities around 10<sup>5</sup> bacterial cells per gram of wet sponge weight, comparable to those of seawater (Hentschel et al., <xref rid=\"B27\" ref-type=\"bibr\">2006</xref>; Gloeckner et al., <xref rid=\"B23\" ref-type=\"bibr\">2014</xref>). Certain bacterial clades are only found in sponges. These &#x201C;sponge-specific clusters&#x201D; are defined as groups containing at least three sequences that: (i) are recovered from different sponge species and/or different geographic locations, (ii) are more closely related to each other than to any other sequence from non-sponge sources, and (iii) cluster together irrespective of the method used (Hentschel et al., <xref rid=\"B26\" ref-type=\"bibr\">2002</xref>). Sponge-specific clusters of bacteria should be regarded as integral parts of the sponge, and these are generally harbored by HMA sponges, whilst LMA sponge species are thought to have less specifically associated microbiota (Webster et al., <xref rid=\"B65\" ref-type=\"bibr\">2004</xref>; Giles et al., <xref rid=\"B22\" ref-type=\"bibr\">2013</xref>), as well as a lower diversity of associated microorganisms. However, deeper sequencing has recently revealed that bacteria previously thought to be confined to sponges may also be present in seawater at low abundances (Taylor et al., <xref rid=\"B55\" ref-type=\"bibr\">2013</xref>). This could be the product of the liberation of bacteria upon the death of sponge cells, or could be caused by a broader distribution of bacteria that survive in the water, but are enriched specifically within the sponge hosts.</p>\n      <p>Sponge-associated archaeal communities appear to follow the HMA-LMA dichotomy found in Bacteria, with typically up to four orders of magnitude higher numbers in HMA over LMA sponges (Bayer et al., <xref rid=\"B4\" ref-type=\"bibr\">2014</xref>). In sponges, the domain Archaea is represented by few specific clades, mainly related to <italic>Cenarcharchaeum symbiosum</italic> (phylum Thaumarchaeota). This phylum dominates sponge-associated microbial communities in sponges from the Arctic (Pape et al., <xref rid=\"B42\" ref-type=\"bibr\">2006</xref>) and Irish deep-sea environments (Jackson et al., <xref rid=\"B29\" ref-type=\"bibr\">2013</xref>). In addition, LMA sponges in some cases harbor as many Archaea as HMA sponges, as is the case for <italic>Stylissa carteri</italic> from the Red Sea (Bayer et al., <xref rid=\"B4\" ref-type=\"bibr\">2014</xref>).</p>\n      <p>Marine eukaryotic microorganisms are both diverse and important components of ocean ecosystems; however, they are understudied compared to other microorganisms. In sponges, most of the fungal diversity has been found to comprise yeast-like organisms. In contrast to the bacterial communities associated with sponges, the existence of sponge-specific fungi is debatable (Rodr&#xED;guez-Marconi et al., <xref rid=\"B45\" ref-type=\"bibr\">2015</xref>; Naim et al., <xref rid=\"B39\" ref-type=\"bibr\">2017</xref>). Other microbial eukaryotes such as the &#x201C;SAR&#x201D; supergroup&#x2014;a clade that includes stramenopiles, alveolates and Rhizaria (Burki et al., <xref rid=\"B8\" ref-type=\"bibr\">2007</xref>)&#x2014;have been encountered in sponges (Webster et al., <xref rid=\"B65\" ref-type=\"bibr\">2004</xref>; Sipkema and Blanch, <xref rid=\"B51\" ref-type=\"bibr\">2009</xref>). However, such micro-eukaryotic communities do not seem to associate specifically with sponges (Rodr&#xED;guez-Marconi et al., <xref rid=\"B45\" ref-type=\"bibr\">2015</xref>).</p>\n      <p>The total microbial diversity, including Bacteria, Archaea and microbial eukaryotes of the same sponge species has very rarely been studied. Moreover, little is understood about the specificity of the microbial associations within a single sponge genus across different geographic locations. In this study, we investigated the microbial diversity of the LMA sponge species <italic>Dysidea avara</italic> and <italic>D. etheria</italic> and compared it with that of the HMA sponge species <italic>Aplysina aerophoba and A. cauliformis</italic>. We hypothesized that the diversity and community structure of bacterial and archaeal microbial communities of the HMA genus <italic>Aplysina</italic> would be better preserved across geographic locations than those of the LMA genus <italic>Dysidea</italic> and that the eukaryotic communities would not be preserved across geographic locations in either genus. To investigate host specificity and to identify intra- and inter-species variation, we sampled individuals of all sponge species in triplicate, as well as seawater from both geographic locations.</p>\n    </sec>\n    <sec sec-type=\"materials and methods\" id=\"s2\">\n      <title>Materials and methods</title>\n      <p>In the Mediterranean Sea, sponge samples were collected by SCUBA diving in April 2014 in Cala Montg&#xF3;, Spain (42&#xB0;06&#x2032; 52.6&#x2033; N, 3&#xB0;10&#x2032; 02.0&#x2033; E). Specimens of <italic>A. aerophoba</italic> were collected from 7.8 to 12.7 m depth, and specimens of <italic>D. avara</italic> at 7.2 m depth. Individual samples were collected in 50 mL sterile tubes after <italic>in situ</italic> identification (Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM6\">1</xref>). Habitat seawater was also collected at the same location in a sterile container to compare sponge-associated bacteria to bacterioplankton. Samples were transported on ice, and approximately 2 h after sampling upon arrival to the laboratory at Centre d'Estudis Avan&#xE7;ats de Blanes (Spain), sponges were washed three times with sterile artificial seawater. Then, 1 cm<sup>3</sup> pieces were cut from each individual and stored in RNALater (Ambion) until DNA isolation. At the Florida Keys in the Caribbean, the same sampling methods were applied in May 2014. <italic>D. etheria</italic> individuals were collected in Summerland Key (24&#xB0; 39&#x2032; 36&#x2033; N, 81&#xB0; 27&#x2032; 36&#x2033; W), and <italic>A. cauliformis</italic> samples were collected in Long Key (24&#xB0; 49&#x2032; 48&#x2033; N, 80&#xB0; 46&#x2032; 12&#x2033; W).</p>\n      <sec>\n        <title>DNA extractions, PCR amplifications and sequencing</title>\n        <p>Total genomic DNA was extracted in triplicate from sponge tissues (150&#x2013;200 mg) and seawater using the FastDNA&#xAE; Spin Kit for Soil (Q-Biogene, Carlsbad, CA) according to the protocol of the manufacturer. The sponge tissue was sampled including tissue from both endodermal and ectodermal tissues. For the isolation of DNA from planktonic cells, 3 L of seawater were filtered onto a 0.22 &#x3BC;m polycarbonate filter (Millipore) and total genomic DNA was extracted from the filter. In the Caribbean Sea, 500 mL of seawater were filtered. The concentration of extracted DNA was determined with a Nanodrop 1000 spectrophotometer (Nanodrop Technologies, Wilmington, DE), whereas its integrity was visually examined by gel electrophoresis on a 1% (w/v) agarose gel stained with ethidium bromide. The extracted DNA was dissolved in TE buffer and stored at &#x2212;20&#xB0;C until further analysis.</p>\n        <p>Amplicon sequencing of phylogenetic marker genes was conducted for all three individuals of <italic>A. aerophoba, A. cauliformis, D. avara</italic>, and <italic>D. etheria</italic>, as well as for seawater. We used Bacteria and Archaea-specific 16S rRNA gene primers published previously (Klindworth et al., <xref rid=\"B31\" ref-type=\"bibr\">2013</xref>). Archaea&#x2014;specific primers amplified a 492 bp DNA sequence containing the V5-V6 regions of the rRNA gene using primers Arch934F: 5&#x2032;-AGGAATTGGCGGGGGAGCA-3&#x2032; and UA1406R: 5&#x2032;-ACGGGCGGTGTGTRCAA-3&#x2032;. For Bacteria, the V3-V4 region of the 16S rRNA gene was targeted and a 428 bp PCR product was obtained with primers 341F: 5&#x2032;-CCTAYGGGRBGCASCAG-3&#x2032; and 806R: 5&#x2032;-GGACTACNNGGGTATCTAAT-3&#x2032;. For characterizing the fungal communities, we amplified the ITS2 region using primers ITS3 (5&#x2032;-GCATCGATGAAGAACGCAGC-3&#x2032;) and ITS4 (5&#x2032;-TCCTCCGCTTATTGATATGC-3&#x2032;) (Liu et al., <xref rid=\"B37\" ref-type=\"bibr\">2012</xref>), suggested as the standard for fungal community analyses (Bates et al., <xref rid=\"B3\" ref-type=\"bibr\">2013</xref>). The ITS region, however, is known to evolve rapidly and be variable in size (Bates et al., <xref rid=\"B3\" ref-type=\"bibr\">2013</xref>), therefore we selected fragments below 500 bp by excision from gel. Finally, we used a 500-bp 18S rRNA gene fragment to characterize unicellular Eukaryota, including, but not restricted to, Fungi. The primers used, Euk1Af (5-CTGGTTGATCCTGCCAG-3) and Euk516r (5-ACCAGACTTGCCCTCC-3) were selected based on previous studies of marine unicellular eukaryotes (D&#xEC;ez et al., <xref rid=\"B15\" ref-type=\"bibr\">2001</xref>; Wilms et al., <xref rid=\"B67\" ref-type=\"bibr\">2006</xref>).</p>\n        <p>PCR amplifications were performed in triplicate for each DNA extraction in a volume of 25 &#x3BC;l containing 1 &#xD7; MyTaq<sup>TM</sup>BIOLINE buffer, 2.5 mM magnesium chloride, 0.25 mM dNTP mixture, 0.5 U MyTaq<sup>TM</sup>BIOLINE DNA polymerase (BIOLINE), 0.5 &#x3BC;M of each primer and 20 ng template DNA. PCR was performed using an initial denaturation at 94&#xB0;C for 2 min, followed by 30 cycles of denaturation at 94&#xB0;C for 30s, annealing at 55&#xB0;C for 30s (except Bacteria at 45&#xB0;C), elongation at 72&#xB0;C for 1 min, and a final elongation at 72&#xB0;C for 5 min. Amplification products were checked on a 1% (w/v) agarose gel. PCR-products were purified using the Wizard&#xAE; SV Gel and PCR Clean-Up System (Promega, Wisconsin, USA). Subsequently, PCR products were quantified using the Quant- iTdsDNA high-sensitivity assay kit (Invitrogen, Grand Island, NY).</p>\n        <p>The PCR products from each sample and each separate microbial community were barcoded prior to sequencing. Libraries for amplicon sequencing were prepared using the NEB Ultra DNA Library Prep Kit for Illumina (NEB, Ipswich, MA, USA) and sequencing was performed on the MiSeq Desktop Sequencer (llumina Inc., San Diego, CA, USA) for 300 cycles in both directions. In total, six MiSeq lanes were sequenced. Bacterial amplifications from Caribbean seawater samples were unsuccessful and therefore removed from further analyses. Sequence data have been deposited in the European Nucleotide Archive under accession number <ext-link ext-link-type=\"DDBJ/EMBL/GenBank\" xlink:href=\"PRJEB22033\">PRJEB22033</ext-link>.</p>\n      </sec>\n      <sec>\n        <title>Sequence processing, taxonomic assignment and diversity analyses</title>\n        <p>To perform a hierarchical taxonomic rank assignment for each sequence, a pipeline developed by the Leibniz-Institut DSMZ (Esposito et al., <xref rid=\"B21\" ref-type=\"bibr\">2015</xref>) was implemented to pre-process raw sequences. Only sequences that contained intact barcode and primer sequences were included in the analyses. Additionally, sequences containing ambiguous bases were removed from the analyses. A quality filter with a Phred score of 20 was applied. After trimming, quality-filtering and pairing the raw reads, 29,064,287 sequences for Bacteria and Archaea and 8,009,549 sequences were obtained for eukaryotes including Fungi. Bacterial and Archaeal sequences were checked for potential chimeras using UCHIME (Edgar et al., <xref rid=\"B17\" ref-type=\"bibr\">2011</xref>). A taxonomic-independent analysis was conducted for all four communities. Sequences that passed the pre-processing steps were first binned into operational taxonomic units (OTUs) using the open-reference pipeline for OTU picking (Caporaso et al., <xref rid=\"B10\" ref-type=\"bibr\">2010b</xref>) at 97% similarity using UCLUST (Edgar, <xref rid=\"B16\" ref-type=\"bibr\">2010</xref>), followed by picking a representative sequence for each OTU. Representative sequences were aligned to the Greengenes Core reference alignment (DeSantis et al., <xref rid=\"B14\" ref-type=\"bibr\">2006</xref>) using PyNAST (Caporaso et al., <xref rid=\"B9\" ref-type=\"bibr\">2010a</xref>), and the resulting sequences were kept for building the OTU tables. Then, OTUs were taxonomically assigned using the Ribosomal Data Project (RDP) Classifier (Cole et al., <xref rid=\"B11\" ref-type=\"bibr\">2009</xref>) command line version 2.10.1. Sequences were considered as classified at the deepest taxonomic rank with a minimum confidence value of 0.5. Taxonomic assignment of the reads showed that bacterial primers also amplified archaeal phyla. These archaeal reads were discarded for the bacterial OTU analyses and all the bacterial reads for the archaeal OTU analyses.</p>\n        <p>High-quality fungal ITS sequencing data were processed using the same pipeline. Chimeric sequences were removed using a combination of reference-based and de novo chimera detection with USEARCH 8 (Edgar, <xref rid=\"B16\" ref-type=\"bibr\">2010</xref>); only reads detected as chimeric with both approaches were discarded. For the subsequent analyses, the ITS2 region was extracted from each sequence using Fungal ITS extractor (Nilsson et al., <xref rid=\"B40\" ref-type=\"bibr\">2010</xref>) (<ext-link ext-link-type=\"uri\" xlink:href=\"http://microbiology.se/software/itsx/\">http://microbiology.se/software/itsx/</ext-link>), as the conserved flanking regions are known to distort similarity searches, taxonomic assignments and clustering results (Bruns and Shefferson, <xref rid=\"B6\" ref-type=\"bibr\">2004</xref>). The resulting ITS2 reads were clustered to operational taxonomic units (OTUs) by UCLUST (Edgar, <xref rid=\"B16\" ref-type=\"bibr\">2010</xref>), with a 100% similarity threshold to reference sequences. As the reference database, we used the UNITE+INSDC data sets, accessed 22 February 2015 (Abarenkov et al., <xref rid=\"B1\" ref-type=\"bibr\">2010</xref>). Singletons (clusters containing only one read) were discarded as they are indistinguishable from sequencing errors (Kunin et al., <xref rid=\"B33\" ref-type=\"bibr\">2010</xref>). OTUs were identified taxonomically by a local BLAST search against the UNITE+INSDC data set using blast-2.2.30 (<ext-link ext-link-type=\"uri\" xlink:href=\"http://blast.ncbi.nlm.nih.gov/Blast.cgi\">http://blast.ncbi.nlm.nih.gov/Blast.cgi</ext-link>) with default settings.</p>\n        <p>High-quality eukaryotic sequences were taxonomically assigned using SILVA (Pruesse et al., <xref rid=\"B43\" ref-type=\"bibr\">2007</xref>), aligned against the SILVA database release 108 (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.arb-silva.de/documentation/background/release-108\">http://www.arb-silva.de/documentation/background/release-108</ext-link>). Non-aligned sequences were removed from the analysis. Subsequently, remaining sequences were dereplicated, clustered and classified. Dereplication, where identical reads are ignored, was performed using custom Python scripts (available at <ext-link ext-link-type=\"uri\" xlink:href=\"https://github.com/mchaib/Eukaryote-community.git\">https://github.com/mchaib/Eukaryote-community.git</ext-link>) with an identity criterion of 1.00. Taxonomic classification was performed by a global search against the SILVA SSURef 108 NR dataset (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.arb-silva.de/projects/ssu-ref-nr/\">http://www.arb-silva.de/projects/ssu-ref-nr/</ext-link>) using USEARCH 8 (Edgar, <xref rid=\"B16\" ref-type=\"bibr\">2010</xref>) with default settings.</p>\n      </sec>\n      <sec>\n        <title>Statistical analyses</title>\n        <p>We conducted a taxonomic-independent analysis based on OTUs for all four microbial communities. To normalize the data for each sample, OTU counts were divided by the total counts of all OTUs within that sample and multiplied by 1000, resulting in relative abundance (RA) expressed as per mille. A log<sub>2</sub> transformation (log<sub>2</sub>(RA + 1)) of the data was performed for statistical comparisons. We used an inclusion threshold &gt;0.1% relative abundance for an OTU in at least one sample. Normalization, transformation and thresholding were performed using R scripts modified from Bulgarelli et al. (<xref rid=\"B7\" ref-type=\"bibr\">2012</xref>). To identify differentially abundant OTUs in the microbial communities between the tested habitats we used linear statistics on relative abundance values (log<sub>2</sub>-transformed, &gt;1% threshold, rarefied) within the R package &#x201C;limma&#x201D;. For each OTU, we fitted a linear model with the habitat as explanatory variable. Differentially abundant OTUs between two habitats were calculated using moderated <italic>t</italic>-tests, an approach suited when the number of measurements per sample is large but sample sizes are small. The resulting <italic>p</italic>-values were adjusted for multiple hypothesis testing using the Benjamini-Hochberg false-discovery-rate (FDR) correction method, where significant enrichment was determined at <italic>p</italic>-values &#x2264; 0.05. To detect OTUs that were more abundant (enriched) in one habitat (sponge or seawater) compared to the other habitats, the intersection of corresponding pairwise comparisons was taken (as described by Bulgarelli et al., <xref rid=\"B7\" ref-type=\"bibr\">2012</xref>).</p>\n        <p>Log<sub>2</sub> transformed relative abundance values were used to calculate a Bray-Curtis dissimilarity matrix using the function &#x201C;vegdist&#x201D; of &#x201C;Vegan.&#x201D; This matrix was used to generate dendrograms using &#x201C;hclust&#x201D; within &#x201C;Vegan&#x201D;, specifying the average clustering method. Variation in composition among sites was assessed using non-metric multidimensional scaling (NMDS) based on the Bray-Curtis dissimilarity matrix. The NMDS plots were generated using the metaMDS() function from the vegan package v 2.4-2. Prior to the PCO, the raw data were log (x+1) transformed and used to produce a distance matrix with the index with the vegdist() function in vegan. The procrustes() function in vegan was used to assess congruence among bacterial and microeukaryotic PCO ordinations. Default values were used for the arguments in the procrustes() analysis. In addition to the procrustes() function, the protest() function in vegan was used to estimate the significance of the Procrustes statistic. The number of permutations in the protest() function was set to 999. The R vegan adonis() function for permutational multivariate analysis of variance (PERMANOVA) was used to test for significant variations in composition between sample groups. In the analysis, the Bray-Curtis distance matrix of OTU composition was the response variable with samples as independent variables. The statistical significance of the <italic>F</italic>-test was assessed with 999 permutations.</p>\n      </sec>\n      <sec>\n        <title>Identification of sponge-specific and sponge/coral-specific clusters</title>\n        <p>Representative sequences of OTUs identified as enriched in a particular sponge host were taxonomically assigned using a BLAST (Altschul et al., <xref rid=\"B2\" ref-type=\"bibr\">1997</xref>) search against the curated ARB-SILVA database containing 178 previously-identified sponge-specific clusters (SC) and 32 sponge/coral-specific clusters (SCC), kindly provided by the authors of that study (Simister et al., <xref rid=\"B49\" ref-type=\"bibr\">2012</xref>). An OTU was only assigned to a cluster if it was more similar to the members of that cluster than to sequences outside the cluster and its similarity to the most similar sequence within that cluster was above 75% (as described in Thomas et al., <xref rid=\"B56\" ref-type=\"bibr\">2016</xref>).</p>\n      </sec>\n    </sec>\n    <sec sec-type=\"results\" id=\"s3\">\n      <title>Results</title>\n      <sec>\n        <title>Variation in bacterial community composition</title>\n        <p>To compare the bacterial and archaeal community structures, we calculated Bray&#x2013;Curtis distances at OTU level and generated NMDS ordination plots (Figures <xref ref-type=\"fig\" rid=\"F1\">1A,B</xref>). For associated Bacteria, the analysis clearly showed an effect of the host sponge species, with all samples from a given sponge species clustering together. <italic>Aplysina</italic> samples from both geographic locations were clearly separated along the first (horizontal) axis of variation from the rest of the samples (Figure <xref ref-type=\"fig\" rid=\"F1\">1A</xref>). Along the second axis of variation, there was a strong separation between (Mediterranean) seawater samples and all sponge samples, as well as separations between the different sponge species. adonis analysis revealed a significant difference in bacterial composition between sponge species (adonis: <italic>F</italic> = 3.334, <italic>R</italic><sup>2</sup> = 0.571, <italic>P</italic> = 0.01), genus (adonis: <italic>F</italic> = 1.427, <italic>R</italic><sup>2</sup> = 0.364, <italic>P</italic> = 0.01), and geographic location (adonis: <italic>F</italic> = 2.521, <italic>R</italic><sup>2</sup> = 0.109, <italic>P</italic> = 0.05). Similarly, hierarchical clustering analyses showed bacteria from each sponge species and genus to cluster separately (Supplementary Figure <xref ref-type=\"supplementary-material\" rid=\"SM1\">1A</xref>).</p>\n        <fig id=\"F1\" position=\"float\">\n          <label>Figure 1</label>\n          <caption>\n            <p>Community structure in prokaryotic communities among all sampled habitats. Ordination showing the first two axes of the NMDS analysis for <bold>(A)</bold> bacterial OTU composition; <bold>(B)</bold> archaeal OTU composition and <bold>(C)</bold> Procrustes analysis comparing bacterial <bold>(A)</bold> and archaeal <bold>(B)</bold> OTU composition. In Procrustes analysis, the arrows point to the target configuration (bacterial OTU composition), the symbols represent the rotated configuration (archaeal composition). Correlation (Corr: 0.48) and significance values (P 0.05) were calculated using the protest function from the vegan R package. Sample groups correspond to <italic>Aplysina aerophoba, A. cauliformis, Dysidea avara, D. etheria</italic> and Mediterranean and Caribbean seawater. Colors for each sample group are specified in the color legend.</p>\n          </caption>\n          <graphic xlink:href=\"fmicb-08-02560-g0001\"/>\n        </fig>\n        <p>The highest bacterial phylum-level diversity was observed in <italic>A. aerophoba</italic>, with 36 different bacterial phyla, followed by Mediterranean seawater with 31 bacterial phyla, and <italic>A. cauliformis</italic> with 27 different phyla. <italic>D. avara</italic> and <italic>D. etheria</italic> had the lowest diversity with 22 and 25 phyla, respectively. At the OTU level, sequences were rarefied to a minimum depth of 8,450 reads per sample (lowest number of reads in a sample) after clustering into 5,044 OTUs (Table <xref ref-type=\"table\" rid=\"T1\">1</xref>, Supplementary Tables <xref ref-type=\"supplementary-material\" rid=\"SM7\">2</xref>, <xref ref-type=\"supplementary-material\" rid=\"SM8\">3</xref>). Chao1 values show that &#x3B1;-diversity values in both <italic>Aplysina</italic> host species and the seawater were significantly higher (<italic>F</italic> = 20.35, <italic>P</italic> = 8.55e&#x2212;05) than those observed for <italic>Dysidea</italic> host species, confirming previous findings stating that LMA sponge species have lower microbial abundance and also lower richness, as compared to HMA sponge species. Overall, our results show that a large number of bacterial OTUs cannot be classified confidently at deeper taxonomic categories. In fact, from 5,044 OTUs, many could not be classified further than Class level, with 32.8% of OTUs not classified at order (1,654), 51.8% OTUs not classified to family (2,615), and up to 87.2% not classified to genus (4,400) level.</p>\n        <table-wrap id=\"T1\" position=\"float\">\n          <label>Table 1</label>\n          <caption>\n            <p>Number of filtered reads for each community and observed OTUs in seawater and sponge samples at a 97% sequence similarity threshold for prokaryotes and microeukaryotes and 98.5% sequence similarity threshold for fungi (ITS2).</p>\n          </caption>\n          <table frame=\"hsides\" rules=\"groups\">\n            <thead>\n              <tr>\n                <th valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">\n                  <bold>Sample name</bold>\n                </th>\n                <th valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><bold>Abbrev</bold>.</th>\n                <th valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">\n                  <bold>Sponge genus</bold>\n                </th>\n                <th valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">\n                  <bold>No. of bacterial reads</bold>\n                </th>\n                <th valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">\n                  <bold>No. of archaeal reads</bold>\n                </th>\n                <th valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">\n                  <bold>No. of fungal reads (ITS2)</bold>\n                </th>\n                <th valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">\n                  <bold>Total no. of microeukarya reads</bold>\n                </th>\n                <th valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">\n                  <bold>No. of microeukarya reads not Porifera</bold>\n                </th>\n                <th valign=\"top\" align=\"center\" colspan=\"2\" style=\"border-bottom: thin solid #000000;\" rowspan=\"1\">\n                  <bold>Bacterial reads</bold>\n                </th>\n                <th valign=\"top\" align=\"center\" colspan=\"2\" style=\"border-bottom: thin solid #000000;\" rowspan=\"1\">\n                  <bold>Archaeal reads</bold>\n                </th>\n                <th valign=\"top\" align=\"center\" colspan=\"2\" style=\"border-bottom: thin solid #000000;\" rowspan=\"1\">\n                  <bold>Fungal reads</bold>\n                </th>\n                <th valign=\"top\" align=\"center\" colspan=\"2\" style=\"border-bottom: thin solid #000000;\" rowspan=\"1\">\n                  <bold>Microeukarya reads</bold>\n                </th>\n              </tr>\n              <tr>\n                <th rowspan=\"1\" colspan=\"1\"/>\n                <th rowspan=\"1\" colspan=\"1\"/>\n                <th rowspan=\"1\" colspan=\"1\"/>\n                <th rowspan=\"1\" colspan=\"1\"/>\n                <th rowspan=\"1\" colspan=\"1\"/>\n                <th rowspan=\"1\" colspan=\"1\"/>\n                <th rowspan=\"1\" colspan=\"1\"/>\n                <th rowspan=\"1\" colspan=\"1\"/>\n                <th valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">\n                  <bold>Observed OTUs</bold>\n                </th>\n                <th valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">\n                  <bold>Expected OTUs (Chao1)<sup>a</sup></bold>\n                </th>\n                <th valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">\n                  <bold>Observed OTUs</bold>\n                </th>\n                <th valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">\n                  <bold>Expected OTUs (Chao1)<sup>a</sup></bold>\n                </th>\n                <th valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">\n                  <bold>Observed OTUs</bold>\n                </th>\n                <th valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">\n                  <bold>Expected OTUs (Chao1)<sup>a</sup></bold>\n                </th>\n                <th valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">\n                  <bold>Observed OTUs</bold>\n                </th>\n                <th valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">\n                  <bold>Expected OTUs (Chao1)<sup>a</sup></bold>\n                </th>\n              </tr>\n            </thead>\n            <tbody>\n              <tr>\n                <td valign=\"top\" align=\"left\" colspan=\"16\" rowspan=\"1\">\n                  <bold>Mediterranean Sea</bold>\n                </td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>A. aerophoba</italic> 1</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">Aa1</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">\n                  <italic>Aplysina</italic>\n                </td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">34,209</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">1,789,477</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">279,210</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">395,294</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">2,173</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">1,197</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">12, 425 &#xB1; 227</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">213</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">264 &#xB1; 23</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">135</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">242 &#xB1; 40</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">70</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">100 &#xB1; 16</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>A. aerophob</italic>a 2</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">Aa2</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">\n                  <italic>Aplysina</italic>\n                </td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">41,947</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">1,226,092</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">352,172</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">402,403</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">3,106</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">1,302</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">12, 621 &#xB1; 160</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">160</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">207 &#xB1; 30</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">167</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">237 &#xB1; 24</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">53</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">91 &#xB1; 22</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>A. aerophoba</italic> 3</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">Aa3</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">\n                  <italic>Aplysina</italic>\n                </td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">37,018</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">2,633,717</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">334,216</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">447,478</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">7,626</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">1,226</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">12, 939 &#xB1; 225</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">219</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">248 &#xB1; 12</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">112</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">158 &#xB1; 20</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">70</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">101 &#xB1; 18</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>D. avara</italic> 1</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">Da1</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">\n                  <italic>Dysidea</italic>\n                </td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">46,412</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">2,017,172</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">206,914</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">447,676</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">5,376</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">681</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">3, 397 &#xB1; 69</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">349</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">373 &#xB1; 11</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">193</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">292 &#xB1; 31</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">91</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">107 &#xB1; 9</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>D. avara</italic> 2</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">Da2</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">\n                  <italic>Dysidea</italic>\n                </td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">31,315</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">2,068,560</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">181,295</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">436,855</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">88,166</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">876</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">4, 386 &#xB1; 114</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">235</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">263 &#xB1; 11</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">244</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">358 &#xB1; 32</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">100</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">118 &#xB1; 10</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>D. avara</italic> 3</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">Da3</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">\n                  <italic>Dysidea</italic>\n                </td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">28,120</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">2,010,458</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">142,302</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">464,741</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">9,693</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">1,208</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">4, 032 &#xB1; 63</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">192</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">205 &#xB1; 7</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">27</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">49 &#xB1; 17</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">107</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">130 &#xB1; 13</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>Seawater Mediterranean</italic> 1</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">SWM1</td>\n                <td rowspan=\"1\" colspan=\"1\"/>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">116,282</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">2,625,898</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">224,261</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">535,350</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">499,579</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">719</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">14, 548 &#xB1; 225</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">394</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">401 &#xB1; 6</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">328</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">515 &#xB1; 48</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">159</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">159 &#xB1; 0</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>Seawater Mediterranean</italic> 2</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">SWM2</td>\n                <td rowspan=\"1\" colspan=\"1\"/>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">108,192</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">2,845,300</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">308,227</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">469,621</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">466,658</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">707</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">14, 029 &#xB1; 235</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">390</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">399 &#xB1; 7</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">393</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">575 &#xB1; 41</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">158</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">168 &#xB1; 10</td>\n              </tr>\n              <tr style=\"border-bottom: thin solid #000000;\">\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>Seawater Mediterranean</italic> 3</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">SWM3</td>\n                <td rowspan=\"1\" colspan=\"1\"/>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">130,946</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">3,097,608</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">200,975</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">471,441</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">465,064</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">745</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">14, 543 &#xB1; 192</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">399</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">407 &#xB1; 6</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">338</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">491 &#xB1; 36</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">159</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">159 &#xB1; 1</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" colspan=\"16\" rowspan=\"1\">\n                  <bold>Caribbean Sea</bold>\n                </td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>A. cauliformis</italic> 1</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">Ac1</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">\n                  <italic>Aplysina</italic>\n                </td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">318,385</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">14,984</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">567,278</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">1,196,853</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">83,779</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">1,147</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">14, 942 &#xB1; 232</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">238</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">255 &#xB1; 6</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">103</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">138 &#xB1; 15</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">25</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">31 &#xB1; 7</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>A. cauliformis</italic> 2</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">Ac2</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">\n                  <italic>Aplysina</italic>\n                </td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">385,282</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">18,047</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">507,567</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">835,488</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">417,744</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">1,226</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">15, 639 &#xB1; 231</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">367</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">374 &#xB1; 3</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">87</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">178 &#xB1; 42</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">27</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">27 &#xB1; 0</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>A. cauliformis</italic> 3</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">Ac3</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">\n                  <italic>Aplysina</italic>\n                </td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">1,493,750</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">249,426</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">9,971</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">149,481</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">8,969</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">781</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">17, 258 &#xB1; 109</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">370</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">452 &#xB1; 23</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">72</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">126 &#xB1; 28</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">86</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">104 &#xB1; 13</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>D. etheria</italic> 1</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">De1</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">\n                  <italic>Dysidea</italic>\n                </td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">8,450</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">84,252</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">160,818</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">155,462</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">137,611</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">837</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">1, 980 &#xB1; 90</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">292</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">309 &#xB1; 9</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">77</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">146 &#xB1; 29</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">49</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">55 &#xB1; 5</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>D. etheria</italic> 2</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">De2</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">\n                  <italic>Dysidea</italic>\n                </td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">93,116</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">5,677</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">229,666</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">123,956</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">44,040</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">832</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">4, 031 &#xB1; 18</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">268</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">3 &#xB1; 2</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">77</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">114 &#xB1; 16</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">65</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">73 &#xB1; 6</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>D. etheria</italic> 3</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">De3</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">\n                  <italic>Dysidea</italic>\n                </td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">15,176</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">45,848</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">192,127</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">131,040</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">13,050</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">866</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">2, 056 &#xB1; 87</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">2</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">305 &#xB1; 17</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">100</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">223 &#xB1; 48</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">62</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">73 &#xB1; 6</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>Seawater Caribbean</italic> 1</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">SWC1</td>\n                <td rowspan=\"1\" colspan=\"1\"/>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">0</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">2,300,482</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">196</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">350,760</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">334,212</td>\n                <td rowspan=\"1\" colspan=\"1\"/>\n                <td rowspan=\"1\" colspan=\"1\"/>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">542</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">598 &#xB1; 19</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">16</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">26 &#xB1; 11</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">66</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">77 &#xB1; 10</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>Seawater Caribbean</italic> 2</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">SWC2</td>\n                <td rowspan=\"1\" colspan=\"1\"/>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">0</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">1,851,259</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">101,978</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">252,887</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">252,868</td>\n                <td rowspan=\"1\" colspan=\"1\"/>\n                <td rowspan=\"1\" colspan=\"1\"/>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">443</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">501 &#xB1; 19</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">222</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">378 &#xB1; 43</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">30</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">31 &#xB1; 1</td>\n              </tr>\n              <tr>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\"><italic>Seawater Caribbean</italic> 3</td>\n                <td valign=\"top\" align=\"left\" rowspan=\"1\" colspan=\"1\">SWC3</td>\n                <td rowspan=\"1\" colspan=\"1\"/>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">0</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">2,294,583</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">15,629</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">284,003</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">274,362</td>\n                <td rowspan=\"1\" colspan=\"1\"/>\n                <td rowspan=\"1\" colspan=\"1\"/>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">563</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">632 &#xB1; 22</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">62</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">147 &#xB1; 44</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">70</td>\n                <td valign=\"top\" align=\"center\" rowspan=\"1\" colspan=\"1\">73 &#xB1; 3</td>\n              </tr>\n            </tbody>\n          </table>\n        </table-wrap>\n        <p>Even though host sponge species and genus diverged in taxonomic composition, as seen in the top 21 most (relatively) abundant OTUs (Supplementary Figure <xref ref-type=\"supplementary-material\" rid=\"SM2\">2</xref>), they also shared several OTUs. The two <italic>Aplysina</italic> species shared 44 OTUs, assigned at the phylum level to Acidobacteria (19), Proteobacteria (11), Chloroflexi (9), Gemmatimonadetes (3), Bacteroidetes (1), and Deinococcus-Thermus (1), while only two different OTUs were shared between the two <italic>Dysidea</italic> host sponge species. The deepest taxonomic assignment possible for these two shared OTUs was the class Alphaproteobacteria. All OTUs shared by the sponge genera fell into monophyletic clusters of &#x201C;sponge-specific&#x201D; (SC) or &#x201C;sponge- and coral-specific&#x201D; (SCC) 16S rRNA gene sequences, as defined previously by Simister et al. (<xref rid=\"B49\" ref-type=\"bibr\">2012</xref>) (Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM11\">6</xref>). The two OTUs shared by the two <italic>Dysidea</italic> species matched SCC26 within the Alphaproteobacteria using the SC 16S rRNA gene sequence database. However, an inspection using BLASTn to the nr database revealed that these two OTUs are 98% identical to uncultured Proteobacteria present in seawater, as shown by their closest matches HM474893.1 and JX405457.1 (Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM11\">6</xref>).</p>\n        <p>We defined sets of OTUs that correspond to symbionts that were potentially acquired prior to the separation of the lineages leading to the sponge species for each genus studied here, based on the intersection of pair-wise comparisons of OTUs enriched in the different sponges and those in the seawater. At the sponge species level, <italic>A. aerophoba</italic> samples had the largest number of enriched OTUs (438). These belonged to the phyla Chloroflexi (269), Proteobacteria (93), Acidobacteria (28), Actinobacteria (16), Gemmatimonadetes (8), Nitrospirae (7), Spirochaetes (3), Deinococcus-Thermus (1), Bacteroidetes (1) and candidate phyla AncK6 (3), PAUC34f (3), Poribacteria (3) and SBR1093 (2) (fdr &lt; 0.05; Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM9\">4</xref>). Similarly, we identified 111 OTUs enriched in <italic>A. cauliformis</italic> within the Acidobacteria (26), Actinobacteria (26), Proteobacteria (22), Chloroflexi (15), Bacteroidetes (10), Gemmatimonadetes (9), Deinococcus-Thermus (1), Spirochaetes (1), and the candidate phylum TM7 (1). Among the LMA sponges, <italic>D. avara</italic> harbored 188 enriched OTUs that belong to the phylum Proteobacteria, in particular the Alphaproteobacteria orders Rhodobacterales, Rickettsiales and Kilionellales, and 1 OTU assigned to Bacteroidetes. Likewise, significantly enriched OTUs (33) in <italic>D. etheria</italic> belonged mainly to the phylum Proteobacteria and three were Bacteroidetes (fdr &lt; 0.05; Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM10\">5</xref>).</p>\n      </sec>\n      <sec>\n        <title>Variation in archaeal community composition</title>\n        <p>Using the Bacteria-specific primers, only 6% of the reads (192,917 reads) were classified as Archaea, while when using the archaea-specific primers this proportion was 89% (25,666,299) of the high-quality reads. After removal of the bacterial sequences from the archaea-specific primer dataset, the remaining sequences were binned into 65,435 OTUs belonging to 8 different archaeal phyla. The phyla Crenarchaeota, Euryarchaeota, Thaumarchaeota and Woesearchaeota were present in all samples, while Aenigmarchaeota, Aigarchaeota, Diapherotrites and Pacearchaeota were consistently found in low numbers only in the replicates of the Mediterranean seawater samples. After applying the inclusion threshold &gt;0.1% relative abundance for an OTU in at least one sample, only 967 OTUs were kept for subsequent analyses (Supplementary Tables <xref ref-type=\"supplementary-material\" rid=\"SM12\">7</xref>, <xref ref-type=\"supplementary-material\" rid=\"SM13\">8</xref>). Of these, 6 were not classified at the order level, 47 were not classified at family level, and 685 were not classified at the genus level (Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM13\">8</xref>). At the OTU level, sequences were rarefied to a minimum depth of 12,694 reads per sample. Based on Bray&#x2013;Curtis distances at the OTU level, archaeal communities from HMA sponge species clustered together for each of the <italic>Aplysina</italic> species and were distinct from the control seawater, while the LMA samples were more dispersed (Figure <xref ref-type=\"fig\" rid=\"F1\">1B</xref>; Supplementary Figure <xref ref-type=\"supplementary-material\" rid=\"SM1\">1B</xref>). HMA samples were separated from the rest along the first axis of variation. The second axis shows the separation of <italic>A. aerophoba</italic> samples from the rest of the samples. Seawater samples occupy an intermediate position, although the samples from both geographic locations are separated from each other. <italic>Dysidea</italic> samples from both species were the most variable within groups. Based on adonis analyses, significant differences in composition were confirmed for sponge species (adonis: <italic>F</italic> = 7.803, <italic>R</italic><sup>2</sup> = 0.438, <italic>P</italic> = 0.01), genera (adonis: <italic>F</italic> = 8.916, <italic>R</italic><sup>2</sup> = 0.335, <italic>P</italic> = 0.01) and geographic location (adonis: <italic>F</italic> = 5.202, <italic>R</italic><sup>2</sup> = 0.098, <italic>P</italic> = 0.01).</p>\n        <p>The family Cenarchaeaceae was overwhelmingly dominant in the sponge species studied here. However, within this taxon different OTUs were significantly enriched for each host sponge species. For example, in <italic>A. aerophoba</italic>, 41 enriched OTUs were assigned to the family Cenarchaeaceae and 39 of these were further assigned to the genus <italic>Nitrosopumilus</italic>. In <italic>A. cauliformis</italic>, the majority of OTUs were assigned to Cenarchaeaceae (21), and only 10 of these were <italic>Nitrosopumilus</italic> (Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM14\">9</xref>). In <italic>D. avara</italic>, there were 8 Cenarchaeaceae OTUs, and in <italic>D. etheria</italic> two OTUs affiliated to the Phylum Euryarchaeota, class Thermoplasmata (Marine group II) (Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM15\">10</xref>; Supplementary Figure <xref ref-type=\"supplementary-material\" rid=\"SM3\">3</xref>). None of these enriched OTUs, even though they belong to the same family (Cenarchaceae) and some to the same genus (<italic>Nitrosopumilus</italic>), were shared between all sponge species included in this study. In order to determine if there was any congruence between the trends observed from the bacterial and archaeal communities, we compared the ordinations obtained with both datasets using Procrustes analysis. There was a significant congruence between both datasets (Procrustes correlation, <italic>R</italic> = 0.48, <italic>P</italic> = 0.05; Figure <xref ref-type=\"fig\" rid=\"F1\">1C</xref>), most likely driven by the specificity of the prokaryotic community for each sponge type (HMA vs. LMA).</p>\n      </sec>\n      <sec>\n        <title>Fungal community composition</title>\n        <p>After DNA sequence quality filtering and ITS2 extraction, a total of 574,040 sequences (816 OTUs, Supplementary Tables <xref ref-type=\"supplementary-material\" rid=\"SM16\">11</xref>, <xref ref-type=\"supplementary-material\" rid=\"SM17\">12</xref>) was found to be fungal and none of the sequences was detected as chimeric. Fungal community structure based on Bray&#x2013;Curtis distances at the OTU level showed clustering patterns that were distinct for HMA and LMA sponge species (Figure <xref ref-type=\"fig\" rid=\"F2\">2A</xref>). The first axis of variation depicts a separation of the two geographic locations under study. The second axis shows a clear separation along sponge genus. The <italic>A. cauliformis</italic> samples were more spread out than those from its Mediterranean counterpart, but also separated on the second axis to the other Caribbean samples. The <italic>D. etheria</italic> samples appeared in close proximity to each other, but mixed with the Caribbean seawater samples: adonis analyses confirmed a significant difference in composition between sponge genera (adonis: <italic>F</italic> = 37.505, <italic>R</italic><sup>2</sup> = 0.654, <italic>P</italic> = 0.01) and along geography (adonis: <italic>F</italic> = 10.326, <italic>R</italic><sup>2</sup> = 0.090, <italic>P</italic> = 0.01). Fungal community structures revealed clustering by host sponge species and then by sponge genus (Supplementary Figure <xref ref-type=\"supplementary-material\" rid=\"SM1\">1C</xref>), the effect being significant (PERMANOVA, <italic>F</italic> = 37.505; <italic>P</italic> = 0.01). On a third level of clustering, seawater samples from both geographic locations shared several OTUs with <italic>Dysidea</italic>, while the <italic>Aplysina</italic> samples formed a completely separate cluster.</p>\n        <fig id=\"F2\" position=\"float\">\n          <label>Figure 2</label>\n          <caption>\n            <p>Community structure in eukaryotic communities. Ordination showing the first two axes of the NMDS analysis for <bold>(A)</bold> fungal OTU composition; <bold>(B)</bold> microeukaryotic OTU composition. Colors for each sample group are specified in the color legend.</p>\n          </caption>\n          <graphic xlink:href=\"fmicb-08-02560-g0002\"/>\n        </fig>\n        <p>Taxonomic assignments for fungal OTUs revealed substantial novelty. For instance, almost 60% of the sequences could not be identified further than Kingdom, that is, 418 OTUs were not assigned to any fungal division (Supplementary Tables <xref ref-type=\"supplementary-material\" rid=\"SM16\">11</xref>, <xref ref-type=\"supplementary-material\" rid=\"SM17\">12</xref>). Moreover, only 30% of the sequences were classified at family level, and 27% at genus level. Only OTUs enriched in a particular environment were further analyzed. OTUs that fell in the division Ascomycota dominated all sponge samples (Figure <xref ref-type=\"fig\" rid=\"F3\">3</xref>). The classes Sordariomycetes, Eurotiomycetes and Leotiomycetes were found most abundantly in the <italic>Dysidea</italic> samples regardless of geographic location (Figure <xref ref-type=\"fig\" rid=\"F3\">3</xref>). In <italic>Aplysina</italic>, up to 70% of OTUs were also assigned to Ascomycota, however the classes differed between the two species. <italic>A. cauliformis</italic> contained Sordariomycetes, Dothideomycetes and Leotiomycetes, while in <italic>A. aerophoba</italic> unknown Ascomycota were dominant. It is noteworthy that Basidiomycota appeared more in the <italic>Aplysina</italic> than in the <italic>Dysidea</italic> samples and more predominantly in <italic>A. cauliformis</italic>. In the seawater, sequences were mostly assigned to Ascomycota and Zygomycota (Figure <xref ref-type=\"fig\" rid=\"F3\">3</xref>). There were no enriched fungal OTUs at the genus level shared between <italic>Aplysina</italic> species (Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM18\">13</xref>). There was only one enriched OTU shared by both <italic>Dysidea</italic> genera, an unidentified Basidiomycete (Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM19\">14</xref>).</p>\n        <fig id=\"F3\" position=\"float\">\n          <label>Figure 3</label>\n          <caption>\n            <p>Compositional patterns of fungal communities using ITS2. Taxonomic overview of most abundant fungal divisions and classes across sample groups. The taxonomic overview shows class taxonomy where available. Colors for each fungal taxon specified in the color legend (squares). The fungal division for each class is further indicated with a filled circle after the name. Green was used to depict Basidiomycota without taxonomic assignments at a deeper taxonomic level; in a similar way, red was used for Ascomycota, and light brown was used for the subdivision Pezizomycotina.</p>\n          </caption>\n          <graphic xlink:href=\"fmicb-08-02560-g0003\"/>\n        </fig>\n      </sec>\n      <sec>\n        <title>Communities of microbial eukaryotes</title>\n        <p>After DNA sequence quality filtering, 7,550,789 non-chimeric reads were retained. Approximately 37% (2,787,534) of all reads were derived from sponges. The taxonomic classification of the sponge reads in each sample confirmed their host identities. After their removal 4,763,255 high-quality non-sponge eukaryotic sequences remained (3,517 OTUs). Fungi contributed with 470,919 (6.2%) of the reads. After applying a relative abundance threshold of 0.1%, 1,101 OTUs were kept (Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM20\">15</xref>). Members of 94 eukaryotic phyla and approximate phylum-level groups (not formally recognized) were detected in sponges and seawater from both geographic locations (Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM21\">16</xref>). In the case of eukaryotes, 42 OTUs were unclassified at the Phylum level (Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM21\">16</xref>), four of which were among the top 20 most abundant OTUs across samples (Supplementary Figures <xref ref-type=\"supplementary-material\" rid=\"SM4\">4</xref>, <xref ref-type=\"supplementary-material\" rid=\"SM5\">5</xref>).</p>\n        <p>Eukaryotic community structure based on Bray&#x2013;Curtis distances at the OTU level showed a separation of clusters between (Figure <xref ref-type=\"fig\" rid=\"F2\">2B</xref>) HMA and LMA sponges and (Figure <xref ref-type=\"fig\" rid=\"F2\">2B</xref>) seawater samples (Figure <xref ref-type=\"fig\" rid=\"F2\">2B</xref>). This separation is picked up by the first axis of variation. The second axis separates sample origins (sponge species and geographic locations). However, these clustering patterns were not significant, neither for composition between sponge genera (adonis: <italic>F</italic> = 2.210, <italic>R</italic><sup>2</sup> = 0.102, <italic>P</italic> = 0.17) nor along geography (adonis: <italic>F</italic> = 1.745, <italic>R</italic><sup>2</sup> = 0.080, <italic>P</italic> = 0.07). In order to assess the overall similarity of eukaryotic communities associated with the sponges and surrounding seawater in both geographic locations, we performed a hierarchical clustering at the OTU level based on Bray&#x2013;Curtis distance (Supplementary Figure <xref ref-type=\"supplementary-material\" rid=\"SM1\">1D</xref>). This analysis revealed that there was no clear distinction between eukaryotic communities in seawater and those from sponges in either geographic location. Similarly, there was no distinctive clustering of all the specimens per habitat. In addition, there was no indication of geography-driven clustering of sponge-associated eukaryotic communities (Supplementary Figure <xref ref-type=\"supplementary-material\" rid=\"SM5\">5</xref>). Finally, Procrustes analysis showed a moderate congruence between the bacterial and the microeukaryotic sequences (Procrustes correlation, <italic>R</italic> = 0.430), but it was not significant (<italic>P</italic> = 0.136).</p>\n      </sec>\n    </sec>\n    <sec sec-type=\"discussion\" id=\"s4\">\n      <title>Discussion</title>\n      <sec>\n        <title>Bacteria and archaea</title>\n        <p>Bacterial communities associated with sponges are often found to follow a species-specific pattern, that is, (1) different individuals of the same sponge species in the same environment harbor similar bacterial communities, and (2) different sponge species from the same environment possess distinct bacterial communities (Webster et al., <xref rid=\"B66\" ref-type=\"bibr\">2010</xref>; Thomas et al., <xref rid=\"B56\" ref-type=\"bibr\">2016</xref>). In our study, the two HMA and LMA sponge species showed low interspecific variation in their associated bacterial, archaeal and fungal communities, clustering first by host species and then by host genus. Following our experimental design, where the same species in each genus were not present in both the Mediterranean and the Caribbean Sea, the statistical tests of geographic location is confounded with the species effect. We tried to alleviate this issue by performing adonis analyses, reversing the addition of terms to the models such that geographic location was considered first. In all cases where host species had previously been shown to be significant, it outweighed the geography effect. Therefore, we are confident, (consistent) with previous reports&#x2014;at least in the case of Bacteria- that the sponge species effect is a main driver of community composition. It is a matter of debate whether both LMA and HMA sponges are equally capable of retaining their associated microbiota (Vacelet and Donadey, <xref rid=\"B61\" ref-type=\"bibr\">1977</xref>; Kamke et al., <xref rid=\"B30\" ref-type=\"bibr\">2010</xref>; Erwin et al., <xref rid=\"B20\" ref-type=\"bibr\">2011</xref>; Schmitt et al., <xref rid=\"B47\" ref-type=\"bibr\">2012</xref>; Giles et al., <xref rid=\"B22\" ref-type=\"bibr\">2013</xref>). In general, HMA sponges have large and diverse microbial communities that relate to previously identified sponge-derived sequences, while LMA sponges have a microbiota with higher similarity to the seawater microbiota. Here, at 97% sequence similarity, and after removal of OTUs at &lt; 0.1% relative abundance, we report richnesses between 681 OTUs (in <italic>D. avara</italic>) and 1302 OTUs (in A. <italic>aerophoba</italic>) (Table <xref ref-type=\"table\" rid=\"T1\">1</xref>), matching expected differences in richness between LMA and HMA species. We find that although the bacterial communities in the <italic>Dysidea</italic> host species are generally less diverse compared to those in the <italic>Aplysina</italic> species, they are very specific in the communities they harbor, distinctive from the seawater samples (Figures <xref ref-type=\"fig\" rid=\"F1\">1A,B</xref>; Supplementary Figure <xref ref-type=\"supplementary-material\" rid=\"SM1\">1</xref>).</p>\n        <p>The sponge genus <italic>Aplysina</italic> encompasses at least 42 accepted species as currently cataloged by the World Register of Marine Species. We included two of these species in our study, namely <italic>A. aerophoba</italic> and <italic>A. cauliformis</italic>. These are found in different habitats (Mediterranean and Caribbean, respectively). Based on the analyses at the OTU level, our findings are in line with previous reports for 16S rRNA gene clone libraries in other <italic>Aplysina</italic> species, identifying Cyanobacteria, Chloroflexi, Proteobacteria (Alpha- and Gamma-), Actinobacteria, Bacteroidetes and Acidobacteria as the most abundant bacterial phyla (Hardoim et al., <xref rid=\"B25\" ref-type=\"bibr\">2009</xref>; Schmitt et al., <xref rid=\"B46\" ref-type=\"bibr\">2011</xref>; Thomas et al., <xref rid=\"B56\" ref-type=\"bibr\">2016</xref>). The diversity of the microbiomes of LMA sponges in our study is dominated by Proteobacteria, namely Alphaproteobacteria of the Rhodobiaceae family and unclassified Alphaproteobacteria. This finding matches previous work that reported that the phylum Proteobacteria was dominant in other LMA sponges. Examples are <italic>Polymastia</italic> sp. (Kamke et al., <xref rid=\"B30\" ref-type=\"bibr\">2010</xref>), <italic>Hymeniacidon heliophila</italic> (Erwin et al., <xref rid=\"B20\" ref-type=\"bibr\">2011</xref>) and <italic>Crambe crambe</italic> (Crou&#xE9; et al., <xref rid=\"B12\" ref-type=\"bibr\">2013</xref>; &#xD6;zt&#xFC;rk et al., <xref rid=\"B41\" ref-type=\"bibr\">2013</xref>). We found that all bacterial OTUs shared between the <italic>Aplysina</italic> species (44) fall within the SC and/or SCC clusters (Simister et al., <xref rid=\"B49\" ref-type=\"bibr\">2012</xref>) and a further examination using BLAST supported the matches to sponge- and/or coral-associated bacteria (Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM11\">6</xref>). These OTUs most likely constitute symbionts whose ancestors engaged in associations with sponges and corals prior to the divergence of the sponge/coral lineages involved. These are good candidates for further work targeting a functional understanding of the interaction between these (likely) symbiotic microbes and their hosts.</p>\n        <p>Regarding archaea, we did not find the sponge-specific archaeon &#x201C;<italic>Cenarchaeum symbiosum</italic>&#x201D; (Thaumarchaeota) (Hallam et al., <xref rid=\"B24\" ref-type=\"bibr\">2006</xref>) in any of our sponge samples. The complete set of representative sequences of Archaea were checked via BLAST and no hit matched the accession number <ext-link ext-link-type=\"DDBJ/EMBL/GenBank\" xlink:href=\"AF083072\">AF083072</ext-link> which corresponds to the 16S rRNA gene for <italic>C. symbiosum</italic> (data not shown). Within the set of enriched OTUs in all sponge samples, we found a remarkable diversity for the Thaumarchaeota of the family Cenarchaeaceae (609 OTUs), and from its genus <italic>Nitrosopumilus</italic> (267 OTUs) (Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM13\">8</xref>, Supplementary Figure <xref ref-type=\"supplementary-material\" rid=\"SM3\">3</xref>). Different OTUs within the Cenarchaeaceae appeared to drive the separation of the samples, following a host sponge-specific pattern. More OTUs unassigned at the genus level were common in <italic>Aplysina</italic> species, while <italic>Nitrosopumilus</italic> OTUs and further OTUs in the phylum Euryarchaeota were enriched in <italic>Dysidea</italic>, particularly in <italic>D. etheria</italic>. It is likely that the sponge habitat provides a suitable niche for species within the Cenarchaeceae to diversify. Earlier, two other Mediterranean sponges have shown enrichments for <italic>Nitrosopumilus</italic>, namely <italic>Petrosia ficiformis</italic> and <italic>Corticium candelabrum</italic> (Sipkema et al., <xref rid=\"B52\" ref-type=\"bibr\">2015</xref>). Some species in this group are known ammonia oxidizers (K&#xF6;nneke et al., <xref rid=\"B32\" ref-type=\"bibr\">2005</xref>), and it has been suggested that sponge-associated archaea may be involved in ammonia oxidation (Hoffmann et al., <xref rid=\"B28\" ref-type=\"bibr\">2009</xref>) through the enzyme ammonia monooxygenase (amoA), probably performing a detoxification role in the sponge, which excretes ammonia as a waste product (Bell, <xref rid=\"B5\" ref-type=\"bibr\">2008</xref>). Our work emphasizes novel archaeal diversity in sponges, in line with previous indications of uncharacterized archaeal diversity in sponges where taxonomic affiliations of <italic>amoA</italic> archaeal genes were poor, with only some clones affiliated to known ammonia oxidizers such as <italic>Nitrosopumilus maritimus</italic> and <italic>Cenarchaeum symbiosum</italic>, but mostly novel sequences within the Cenarchaeceae (Turque et al., <xref rid=\"B57\" ref-type=\"bibr\">2010</xref>).</p>\n      </sec>\n      <sec>\n        <title>Eukaryotic microorganisms</title>\n        <p>As compared to bacteria and archaea, the patterns and processes of fungal community composition in marine sponges are less well documented. Studies of wood-inhabiting marine fungi have shown that geographic location is the main factor determining fungal community composition (R&#xE4;m&#xE4; et al., <xref rid=\"B44\" ref-type=\"bibr\">2014</xref>). In this study, we found a compositional difference in the fungal communities between sponge and seawater samples (Figure <xref ref-type=\"fig\" rid=\"F3\">3</xref>); however, the structure of the fungal communities is mainly driven by the sponge host and sponge genus (Figure <xref ref-type=\"fig\" rid=\"F2\">2</xref>). The potential for specific interactions between the sponges and their associated fungal communities has been described before. For instance, the vertical transmission of a unicellular fungus via the oocytes in the Mediterranean sponge <italic>Chondrilla nucula</italic> was described based on transmission electron microscopy (TEM) analysis (Maldonado et al., <xref rid=\"B38\" ref-type=\"bibr\">2005</xref>). However, specificity of fungal communities in sponges had not been observed so far following the sponge-specificity concept developed for bacterial communities (Hentschel et al., <xref rid=\"B26\" ref-type=\"bibr\">2002</xref>). Most likely due to the considerable depth of sequencing for fungal communities in this study (Table <xref ref-type=\"table\" rid=\"T1\">1</xref>), we have been able to observe novel fungal diversity (Supplementary Table <xref ref-type=\"supplementary-material\" rid=\"SM17\">12</xref>) and show that this diversity is largely determined by the host species and genus.</p>\n        <p>Using sequencing of a different marker, the V7-V8 region of the 18S rRNA gene, Naim et al. (<xref rid=\"B39\" ref-type=\"bibr\">2017</xref>) found similar fungal groups to be associated with sponges from the North Sea and the Mediterranean Sea, including <italic>A. aerophoba</italic>. Ascomycota clearly dominated the recovered community assigned to the division level. The 18S rRNA gene primers used in this study (Euk1Af and Euk516r) were chosen based on their former performance in broadly targeting the diversity of marine picoeukaryotes (D&#xEC;ez et al., <xref rid=\"B15\" ref-type=\"bibr\">2001</xref>; Wilms et al., <xref rid=\"B67\" ref-type=\"bibr\">2006</xref>). Hence, the amplicons provide descriptive measures of both the fungal community and, more broadly, the eukaryotic community associated with sponges. The majority of 18S rRNA gene sequences was derived from the sponge hosts (on average, 85% of reads). In total, 821,333 eukaryotic reads were not assigned to Porifera in the sponges investigated. Even though we use only 15% of the generated reads, this number exceeds by far that of previous studies in which next generation sequencing technology was used, with respect to microeukaryote diversity in sponges (Simister et al., <xref rid=\"B49\" ref-type=\"bibr\">2012</xref>; Rodr&#xED;guez-Marconi et al., <xref rid=\"B45\" ref-type=\"bibr\">2015</xref>). However, there is a limitation driven by the large data loss for the sponge samples (as reported in Table <xref ref-type=\"table\" rid=\"T1\">1</xref>). The variability in the identified OTUs was large among samples, and within sponges of each species, most OTUs were only present in one or two out of the three replicates per group. Due to the low sample size, it becomes challenging to confidently discern specificity for the groups of microbial eukaryotes based on the 18S rRNA marker gene. Besides an increased sample size for future studies, we suggest designing primers that will preferentially amplify eukaryotes other than Porifera, thus preventing uneven data loss across sampled individuals.</p>\n        <p>A few previous studies also analyzed the 18S rRNA gene sequences obtained from eukaryotic microorganisms living within sponges. These studies reported protists (Alveolata, Mesomycetozoea, Rhizaria), protozoans (Amoebozoa, Apusozoa and Euglenozoa), protophytes (Cryptophyta, Rhodophyta, Viridiplantae, Cnidaria, Annelida and Stramenopiles (Webster et al., <xref rid=\"B65\" ref-type=\"bibr\">2004</xref>; Sipkema and Blanch, <xref rid=\"B51\" ref-type=\"bibr\">2009</xref>; Simister et al., <xref rid=\"B49\" ref-type=\"bibr\">2012</xref>; Li et al., <xref rid=\"B36\" ref-type=\"bibr\">2014</xref>) to be present. However, diatoms, dinoflagellates and fungi fulfilling the criteria for sponge-specific clusters (Simister et al., <xref rid=\"B49\" ref-type=\"bibr\">2012</xref>) were not found in this study. Communities of eukaryotic microorganisms, with the exception of fungi, apparently did not assemble randomly, although no clear host- or genus-specific patterns could be discerned from the current study.</p>\n      </sec>\n    </sec>\n    <sec sec-type=\"conclusions\" id=\"s5\">\n      <title>Conclusion</title>\n      <p>The three-domain microbial communities from the LMA and HMA sponges of the genera <italic>Aplysina</italic> and <italic>Dysidea</italic> studied here are different from the surrounding planktonic communities, expanding previous observations for Bacteria. The correspondence between the low abundance of microorganisms and low phylum-level diversity in the LMA genus <italic>Dysidea</italic> was only true for bacterial communities, not so for microbes in the domains Archaea and Eukarya. We defined sets of bacterial OTUs that are enriched in particular sponge species and shared in each sponge genus, the latter indicating probable microorganisms that establish a symbiosis with each genus before the split of the sponge species studied here. These putative symbionts within each sponge genus are good candidates for future efforts to understand the functional roles that mediate their interactions with the sponge hosts. Furthermore, we present new diversity in the three domains of life. Especially in Bacteria and Fungi, these undescribed taxonomic levels are good candidates for bioprospection of associated microorganisms.</p>\n    </sec>\n    <sec id=\"s6\">\n      <title>Author contributions</title>\n      <p>MC, DS, JO and JDvE initiated and coordinated the project. MC, SH and BB planned and executed the sequencing and analysis pipelines for the project; MC, SH, and BB analyzed its outcome. MC wrote the paper, and all authors contributed to its improvement.</p>\n      <sec>\n        <title>Conflict of interest statement</title>\n        <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p>\n      </sec>\n    </sec>\n  </body>\n  <back>\n    <ack>\n      <p>We thank Shirley Pomponi and Stephanie Munroe from Florida Atlantic University Research Corporation for providing all the samples from the Caribbean Sea used in this study as well as Mar&#xED;a J. Uriz from the Centre d'Estudis Avan&#xE7;ats de Blanes (CEAB), CSIC, Blanes, Spain for providing support for sample collection and access to the laboratories for prompt processing. We would also like to thank the Center for Information Technology of the University of Groningen for their support and providing access to the Peregrine high-performance computing cluster.</p>\n    </ack>\n    <fn-group>\n      <fn fn-type=\"financial-disclosure\">\n        <p><bold>Funding.</bold> The research leading to these results has received funding from the People Programme (Marie Curie Actions) of the European Union's Seventh Framework Programme FP7/2007-2013/ under REA grant agreement n&#xB0; 607786, BluePharmTrain.</p>\n      </fn>\n    </fn-group>\n    <sec sec-type=\"supplementary-material\" id=\"s7\">\n      <title>Supplementary material</title>\n      <p>The Supplementary Material for this article can be found online at: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.frontiersin.org/articles/10.3389/fmicb.2017.02560/full#supplementary-material\">https://www.frontiersin.org/articles/10.3389/fmicb.2017.02560/full#supplementary-material</ext-link></p>\n      <supplementary-material content-type=\"local-data\" id=\"SM1\">\n        <label>Supplementary Figure 1</label>\n        <caption>\n          <p>Community structure for samples assessed for all four microbial communities. Bray-Curtis dissimilarity of samples collected from the Mediterranean and the Caribbean Sea. OTUs with relative abundance &gt; 0.1% were included in the analysis. <bold>(A)</bold> Bacteria, <bold>(B)</bold> Archaea, <bold>(C)</bold> Fungi, <bold>(D)</bold> microbial Eukarya.</p>\n        </caption>\n        <media xlink:href=\"Image1.JPEG\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM2\">\n        <label>Supplementary Figure 2</label>\n        <caption>\n          <p>Bacterial phylum level relative abundance in the studied habitats for the top 19 most abundant phyla. Phyla as specified in color legend.</p>\n        </caption>\n        <media xlink:href=\"Image2.JPEG\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM3\">\n        <label>Supplementary Figure 3</label>\n        <caption>\n          <p>Boxplots depicting Archaea OTUs found as significantly enriched in a given habitat. Log&#x2013;relative normalized abundance per OTU. Colors show different habitats as specified in color legend. Padj: Adjusted <italic>p</italic>-value corrected for multiple comparisons with FDR.</p>\n        </caption>\n        <media xlink:href=\"Image3.JPEG\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM4\">\n        <label>Supplementary Figure 4</label>\n        <caption>\n          <p>Relative abundance for the top 10 abundant phylum-level groups of eukaryotes found in this study, split based on sponge species habitat. Color legens as specified in figure.</p>\n        </caption>\n        <media xlink:href=\"Image4.JPEG\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM5\">\n        <label>Supplementary Figure 5</label>\n        <caption>\n          <p>Heatmap for the top 100 most abundant microbial Eukaryotic OTUs. Bray-Curtis dissimilarity OTUs with relative abundance &gt;0.1% were included in the analysis.</p>\n        </caption>\n        <media xlink:href=\"Image5.JPEG\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM6\">\n        <label>Supplementary Table 1</label>\n        <caption>\n          <p>Metadata associated with samples used in this study.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM7\">\n        <label>Supplementary Table 2</label>\n        <caption>\n          <p>Bacterial relative abundance table including OTUs above 0.1% in at least one sample.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM8\">\n        <label>Supplementary Table 3</label>\n        <caption>\n          <p>Bacterial correspondences between OTU identifiers and assigned taxonomy using RDP.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM9\">\n        <label>Supplementary Table 4</label>\n        <caption>\n          <p>Bacterial OTUs enriched in <italic>Aplysina</italic> species.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM10\">\n        <label>Supplementary Table 5</label>\n        <caption>\n          <p>Bacterial OTUs enriched in <italic>Dysidea</italic> species.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM11\">\n        <label>Supplementary Table 6</label>\n        <caption>\n          <p>Bacterial OTUs enriched in <italic>Aplysina</italic> and <italic>Dysidea</italic> host genera identified as belonging to sponge-specific clusters (SC) and/or sponge/coral-specific clusters (SCC).</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM12\">\n        <label>Supplementary Table 7</label>\n        <caption>\n          <p>Archaeal relative abundance table including OTUs above 0.1% in at least one sample.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM13\">\n        <label>Supplementary Table 8</label>\n        <caption>\n          <p>Archaeal correspondences between OTU identifiers and assigned taxonomy using RDP.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM14\">\n        <label>Supplementary Table 9</label>\n        <caption>\n          <p>Archaeal OTUs enriched in <italic>Aplysina</italic> species.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM15\">\n        <label>Supplementary Table 10</label>\n        <caption>\n          <p>Archaeal OTUs enriched in <italic>Dysidea</italic> species.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM16\">\n        <label>Supplementary Table 11</label>\n        <caption>\n          <p>Fungal relative abundance table including OTUs above 0.1% in at least one sample using ITS2.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM17\">\n        <label>Supplementary Table 12</label>\n        <caption>\n          <p>Fungal correspondences between OTU identifiers and assigned taxonomy using the UNITE database.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM18\">\n        <label>Supplementary Table 13</label>\n        <caption>\n          <p>Fungal OTUs enriched in <italic>Aplysina</italic> species.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM19\">\n        <label>Supplementary Table 14</label>\n        <caption>\n          <p>Fungal OTUs enriched in <italic>Dysidea</italic> species.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM20\">\n        <label>Supplementary Table 15</label>\n        <caption>\n          <p>Microbial eukaryotes relative abundance table including OTUs above 0.1% in at least one sample using 18S rRNA.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n      <supplementary-material content-type=\"local-data\" id=\"SM21\">\n        <label>Supplementary Table 16</label>\n        <caption>\n          <p>Microbial eukaryotes correspondences between OTU identifiers and assigned taxonomy using the SILVA database.</p>\n        </caption>\n        <media xlink:href=\"DataSheet1.xlsx\">\n          <caption>\n            <p>Click here for additional data file.</p>\n          </caption>\n        </media>\n      </supplementary-material>\n    </sec>\n    <ref-list>\n      <title>References</title>\n      <ref id=\"B1\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Abarenkov</surname><given-names>K.</given-names></name><name><surname>Henrik Nilsson</surname><given-names>R.</given-names></name><name><surname>Larsson</surname><given-names>K. H.</given-names></name><name><surname>Alexander</surname><given-names>I. J.</given-names></name><name><surname>Eberhardt</surname><given-names>U.</given-names></name><name><surname>Erland</surname><given-names>S.</given-names></name><etal/></person-group>. (<year>2010</year>). <article-title>The UNITE database for molecular identification of fungi&#x2013;recent updates and future perspectives</article-title>. <source>New Phytol</source>. <volume>186</volume>, <fpage>281</fpage>&#x2013;<lpage>285</lpage>. <pub-id pub-id-type=\"doi\">10.1111/j.1469-8137.2009.03160.x</pub-id><pub-id pub-id-type=\"pmid\">20409185</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B2\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Altschul</surname><given-names>S. F.</given-names></name><name><surname>Madden</surname><given-names>T. L.</given-names></name><name><surname>Sch&#xE4;ffer</surname><given-names>A. A.</given-names></name><name><surname>Zhang</surname><given-names>J.</given-names></name><name><surname>Zhang</surname><given-names>Z.</given-names></name><name><surname>Miller</surname><given-names>W.</given-names></name><etal/></person-group>. (<year>1997</year>). <article-title>Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</article-title>. <source>Nucleic Acids Res</source>. <volume>25</volume>, <fpage>3389</fpage>&#x2013;<lpage>3402</lpage>. <pub-id pub-id-type=\"doi\">10.1093/nar/25.17.3389</pub-id><pub-id pub-id-type=\"pmid\">9254694</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B3\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bates</surname><given-names>S. T.</given-names></name><name><surname>Ahrendt</surname><given-names>S.</given-names></name><name><surname>Bik</surname><given-names>H. M.</given-names></name><name><surname>Bruns</surname><given-names>T. D.</given-names></name><name><surname>Caporaso</surname><given-names>J. G.</given-names></name><name><surname>Cole</surname><given-names>J.</given-names></name><etal/></person-group>. (<year>2013</year>). <article-title>Meeting report: fungal its workshop (October 2012)</article-title>. <source>Stand. Genomic Sci</source>. <volume>8</volume>, <fpage>118</fpage>&#x2013;<lpage>123</lpage>. <pub-id pub-id-type=\"doi\">10.4056/sigs.3737409</pub-id><pub-id pub-id-type=\"pmid\">23961317</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B4\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bayer</surname><given-names>K.</given-names></name><name><surname>Kamke</surname><given-names>J.</given-names></name><name><surname>Hentschel</surname><given-names>U.</given-names></name></person-group> (<year>2014</year>). <article-title>Quantification of bacterial and archaeal symbionts in high and low microbial abundance sponges using real-time PCR</article-title>. <source>FEMS Microbiol. Ecol</source>. <volume>89</volume>, <fpage>679</fpage>&#x2013;<lpage>690</lpage>. <pub-id pub-id-type=\"doi\">10.1111/1574-6941.12369</pub-id><pub-id pub-id-type=\"pmid\">24942664</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B5\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bell</surname><given-names>J. J.</given-names></name></person-group> (<year>2008</year>). <article-title>The functional roles of marine sponges</article-title>. <source>Estuar. Coast. Shelf Sci</source>. <volume>79</volume>, <fpage>341</fpage>&#x2013;<lpage>353</lpage>. <pub-id pub-id-type=\"doi\">10.1016/j.ecss.2008.05.002</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B6\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bruns</surname><given-names>T. D.</given-names></name><name><surname>Shefferson</surname><given-names>R. P.</given-names></name></person-group> (<year>2004</year>). <article-title>Evolutionary studies of ectomycorrhizal fungi: recent advances and future directions</article-title>. <source>Can. J. Bot. Can. Bot</source>. <volume>82</volume>, <fpage>1122</fpage>&#x2013;<lpage>1132</lpage>. <pub-id pub-id-type=\"doi\">10.1139/b04-021</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B7\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bulgarelli</surname><given-names>D.</given-names></name><name><surname>Rott</surname><given-names>M.</given-names></name><name><surname>Schlaeppi</surname><given-names>K.</given-names></name><name><surname>Ver Loren van Themaat</surname><given-names>E.</given-names></name><name><surname>Ahmadinejad</surname><given-names>N.</given-names></name><name><surname>Assenza</surname><given-names>F.</given-names></name><etal/></person-group>. (<year>2012</year>). <article-title>Revealing structure and assembly cues for Arabidopsis root-inhabiting bacterial microbiota</article-title>. <source>Nature</source>\n<volume>488</volume>, <fpage>91</fpage>&#x2013;<lpage>95</lpage>. <pub-id pub-id-type=\"doi\">10.1038/nature11336</pub-id><pub-id pub-id-type=\"pmid\">22859207</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B8\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Burki</surname><given-names>F.</given-names></name><name><surname>Shalchian-Tabrizi</surname><given-names>K.</given-names></name><name><surname>Minge</surname><given-names>M.</given-names></name><name><surname>Skj&#xE6;veland</surname><given-names>&#xC5;.</given-names></name><name><surname>Nikolaev</surname><given-names>S. I.</given-names></name><name><surname>Jakobsen</surname><given-names>K. S.</given-names></name><etal/></person-group>. (<year>2007</year>). <article-title>Phylogenomics reshuffles the eukaryotic supergroups</article-title>. <source>PLoS ONE</source>\n<volume>2</volume>:<fpage>e00790</fpage>. <pub-id pub-id-type=\"doi\">10.1371/journal.pone.0000790</pub-id><pub-id pub-id-type=\"pmid\">17726520</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B9\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Caporaso</surname><given-names>J. G.</given-names></name><name><surname>Bittinger</surname><given-names>K.</given-names></name><name><surname>Bushman</surname><given-names>F. D.</given-names></name><name><surname>DeSantis</surname><given-names>T. Z.</given-names></name><name><surname>Andersen</surname><given-names>G. L.</given-names></name><name><surname>Knight</surname><given-names>R.</given-names></name></person-group> (<year>2010a</year>). <article-title>PyNAST: a flexible tool for aligning sequences to a template alignment</article-title>. <source>Bioinformatics</source>\n<volume>26</volume>, <fpage>266</fpage>&#x2013;<lpage>267</lpage>. <pub-id pub-id-type=\"doi\">10.1093/bioinformatics/btp636</pub-id><pub-id pub-id-type=\"pmid\">19914921</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B10\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Caporaso</surname><given-names>J. G.</given-names></name><name><surname>Kuczynski</surname><given-names>J.</given-names></name><name><surname>Stombaugh</surname><given-names>J.</given-names></name><name><surname>Bittinger</surname><given-names>K.</given-names></name><name><surname>Bushman</surname><given-names>F. D.</given-names></name><name><surname>Costello</surname><given-names>E. K.</given-names></name><etal/></person-group>. (<year>2010b</year>). <article-title>QIIME allows analysis of high-throughput community sequencing data</article-title>. <source>Nat. Methods</source>\n<volume>7</volume>, <fpage>335</fpage>&#x2013;<lpage>336</lpage>. <pub-id pub-id-type=\"doi\">10.1038/nmeth.f.303</pub-id><pub-id pub-id-type=\"pmid\">20383131</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B11\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cole</surname><given-names>J. R.</given-names></name><name><surname>Wang</surname><given-names>Q.</given-names></name><name><surname>Cardenas</surname><given-names>E.</given-names></name><name><surname>Fish</surname><given-names>J.</given-names></name><name><surname>Chai</surname><given-names>B.</given-names></name><name><surname>Farris</surname><given-names>R. J.</given-names></name><etal/></person-group>. (<year>2009</year>). <article-title>The Ribosomal database project: improved alignments and new tools for rRNA analysis</article-title>. <source>Nucleic Acids Res</source>. <volume>37</volume>, <fpage>141</fpage>&#x2013;<lpage>145</lpage>. <pub-id pub-id-type=\"doi\">10.1093/nar/gkn879</pub-id><pub-id pub-id-type=\"pmid\">19004872</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B12\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Crou&#xE9;</surname><given-names>J.</given-names></name><name><surname>West</surname><given-names>N. J.</given-names></name><name><surname>Escande</surname><given-names>M.-L.</given-names></name><name><surname>Intertaglia</surname><given-names>L.</given-names></name><name><surname>Lebaron</surname><given-names>P.</given-names></name><name><surname>Suzuki</surname><given-names>M. T.</given-names></name></person-group> (<year>2013</year>). <article-title>A single betaproteobacterium dominates the microbial community of the crambescidine-containing sponge <italic>Crambe crambe</italic></article-title>. <source>Sci. Rep</source>. <volume>3</volume>:<fpage>2583</fpage>. <pub-id pub-id-type=\"doi\">10.1038/srep02583</pub-id><pub-id pub-id-type=\"pmid\">24002533</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B13\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>De Caralt</surname><given-names>S.</given-names></name><name><surname>Uriz</surname><given-names>M. J.</given-names></name><name><surname>Wijffels</surname><given-names>R. H.</given-names></name></person-group> (<year>2007</year>). <article-title>Vertical transmission and successive location of symbiotic bacteria during embryo development and larva formation in <italic>Corticium candelabrum</italic> (Porifera: Demospongiae)</article-title>. <source>J. Mar. Biol. Assoc. U.K.</source>\n<volume>87</volume>, <fpage>1693</fpage>&#x2013;<lpage>1699</lpage>. <pub-id pub-id-type=\"doi\">10.1017/S0025315407056846</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B14\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>DeSantis</surname><given-names>T. Z.</given-names></name><name><surname>Hugenholtz</surname><given-names>P.</given-names></name><name><surname>Larsen</surname><given-names>N.</given-names></name><name><surname>Rojas</surname><given-names>M.</given-names></name><name><surname>Brodie</surname><given-names>E. L.</given-names></name><name><surname>Keller</surname><given-names>K.</given-names></name><etal/></person-group>. (<year>2006</year>). <article-title>Greengenes, a chimera-checked 16S rRNA gene database and workbench compatible with ARB</article-title>. <source>Appl. Environ. Microbiol</source>. <volume>72</volume>, <fpage>5069</fpage>&#x2013;<lpage>5072</lpage>. <pub-id pub-id-type=\"doi\">10.1128/AEM.03006-05</pub-id><pub-id pub-id-type=\"pmid\">16820507</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B15\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>D&#xEC;ez</surname><given-names>B.</given-names></name><name><surname>Pedr&#xF3;s-Ali&#xF3;</surname><given-names>C.</given-names></name><name><surname>Massana</surname><given-names>R.</given-names></name></person-group> (<year>2001</year>). <article-title>Study of genetic diversity of eukaryotic picoplankton in different oceanic regions by small-subunit rRNA gene cloning and sequencing study of genetic diversity of eukaryotic picoplankton in different oceanic regions by small-subunit rRNA gene cloning and</article-title>. <source>Appl. Environ. Microbiol</source>. <volume>67</volume>, <fpage>2932</fpage>&#x2013;<lpage>2941</lpage>. <pub-id pub-id-type=\"doi\">10.1128/AEM.67.7.2932-2941.2001</pub-id><pub-id pub-id-type=\"pmid\">11425705</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B16\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Edgar</surname><given-names>R. C.</given-names></name></person-group> (<year>2010</year>). <article-title>Search and clustering orders of magnitude faster than BLAST</article-title>. <source>Bioinformatics</source>\n<volume>26</volume>, <fpage>2460</fpage>&#x2013;<lpage>2461</lpage>. <pub-id pub-id-type=\"doi\">10.1093/bioinformatics/btq461</pub-id><pub-id pub-id-type=\"pmid\">20709691</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B17\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Edgar</surname><given-names>R. C.</given-names></name><name><surname>Haas</surname><given-names>B. J.</given-names></name><name><surname>Clemente</surname><given-names>J. C.</given-names></name><name><surname>Quince</surname><given-names>C.</given-names></name><name><surname>Knight</surname><given-names>R.</given-names></name></person-group> (<year>2011</year>). <article-title>UCHIME improves sensitivity and speed of chimera detection</article-title>. <source>Bioinformatics</source>\n<volume>27</volume>, <fpage>2194</fpage>&#x2013;<lpage>2200</lpage>. <pub-id pub-id-type=\"doi\">10.1093/bioinformatics/btr381</pub-id><pub-id pub-id-type=\"pmid\">21700674</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B18\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Enticknap</surname><given-names>J. J.</given-names></name><name><surname>Kelly</surname><given-names>M.</given-names></name><name><surname>Peraud</surname><given-names>O.</given-names></name><name><surname>Hill</surname><given-names>R. T.</given-names></name></person-group> (<year>2006</year>). <article-title>Characterization of a culturable alphaproteobacterial symbiont common to many marine sponges and evidence for vertical transmission via sponge larvae</article-title>. <source>Appl. Environ. Microbiol</source>. <volume>72</volume>, <fpage>3724</fpage>&#x2013;<lpage>3732</lpage>. <pub-id pub-id-type=\"doi\">10.1128/AEM.72.5.3724-3732.2006</pub-id><pub-id pub-id-type=\"pmid\">16672523</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B19\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ereskovsky</surname><given-names>A. V.</given-names></name><name><surname>Gonobobleva</surname><given-names>E.</given-names></name><name><surname>Vishnyakov</surname><given-names>A.</given-names></name></person-group> (<year>2005</year>). <article-title>Morphological evidence for vertical transmission of symbiotic bacteria in the viviparous sponge <italic>Halisarca dujardini</italic> johnston (Porifera, Demospongiae, Halisarcida)</article-title>. <source>Mar. Biol</source>. <volume>146</volume>, <fpage>869</fpage>&#x2013;<lpage>875</lpage>. <pub-id pub-id-type=\"doi\">10.1007/s00227-004-1489-1</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B20\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Erwin</surname><given-names>P. M.</given-names></name><name><surname>Olson</surname><given-names>J. B.</given-names></name><name><surname>Thacker</surname><given-names>R. W.</given-names></name></person-group> (<year>2011</year>). <article-title>Phylogenetic diversity, host-specificity and community profiling of sponge-associated bacteria in the northern gulf of mexico</article-title>. <source>PLoS ONE</source>\n<volume>6</volume>:<fpage>e26806</fpage>. <pub-id pub-id-type=\"doi\">10.1371/journal.pone.0026806</pub-id><pub-id pub-id-type=\"pmid\">22073197</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B21\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Esposito</surname><given-names>A.</given-names></name><name><surname>Ahmed</surname><given-names>E.</given-names></name><name><surname>Ciccazzo</surname><given-names>S.</given-names></name><name><surname>Sikorski</surname><given-names>J.</given-names></name><name><surname>Overmann</surname><given-names>J.</given-names></name><name><surname>Holmstr&#xF6;m</surname><given-names>S. J. M.</given-names></name><etal/></person-group>. (<year>2015</year>). <article-title>Comparison of rock varnish bacterial communities with surrounding non-varnished rock surfaces: taxon-specific analysis and morphological description</article-title>. <source>Microb. Ecol</source>. <volume>70</volume>, <fpage>741</fpage>&#x2013;<lpage>750</lpage>. <pub-id pub-id-type=\"doi\">10.1007/s00248-015-0617-4</pub-id><pub-id pub-id-type=\"pmid\">25921518</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B22\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Giles</surname><given-names>E. C.</given-names></name><name><surname>Kamke</surname><given-names>J.</given-names></name><name><surname>Moitinho-Silva</surname><given-names>L.</given-names></name><name><surname>Taylor</surname><given-names>M. W.</given-names></name><name><surname>Hentschel</surname><given-names>U.</given-names></name><name><surname>Ravasi</surname><given-names>T.</given-names></name><etal/></person-group>. (<year>2013</year>). <article-title>Bacterial community profiles in low microbial abundance sponges</article-title>. <source>FEMS Microbiol. Ecol</source>. <volume>83</volume>, <fpage>232</fpage>&#x2013;<lpage>241</lpage>. <pub-id pub-id-type=\"doi\">10.1111/j.1574-6941.2012.01467.x</pub-id><pub-id pub-id-type=\"pmid\">22882238</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B23\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gloeckner</surname><given-names>V.</given-names></name><name><surname>Wehrl</surname><given-names>M.</given-names></name><name><surname>Moitinho-Silva</surname><given-names>L.</given-names></name><name><surname>Gernert</surname><given-names>C.</given-names></name><name><surname>Schupp</surname><given-names>P.</given-names></name><name><surname>Pawlik</surname><given-names>J. R.</given-names></name><etal/></person-group>. (<year>2014</year>). <article-title>The HMA-LMA dichotomy revisited: an electron microscopical survey of 56 sponge species</article-title>. <source>Biol. Bull</source>. <volume>227</volume>, <fpage>78</fpage>&#x2013;<lpage>88</lpage>. <pub-id pub-id-type=\"doi\">10.1086/BBLv227n1p78</pub-id><pub-id pub-id-type=\"pmid\">25216505</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B24\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hallam</surname><given-names>S. J.</given-names></name><name><surname>Konstantinidis</surname><given-names>K. T.</given-names></name><name><surname>Putnam</surname><given-names>N.</given-names></name><name><surname>Schleper</surname><given-names>C.</given-names></name><name><surname>Watanabe</surname><given-names>Y.</given-names></name><name><surname>Sugahara</surname><given-names>J.</given-names></name><etal/></person-group>. (<year>2006</year>). <article-title>Genomic analysis of the uncultivated marine crenarchaeote Cenarchaeum symbiosum</article-title>. <source>Proc. Natl. Acad. Sci. U.S.A</source>. <volume>103</volume>, <fpage>18296</fpage>&#x2013;<lpage>18301</lpage>. <pub-id pub-id-type=\"doi\">10.1073/pnas.0608549103</pub-id><pub-id pub-id-type=\"pmid\">17114289</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B25\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hardoim</surname><given-names>C. C.</given-names></name><name><surname>Costa</surname><given-names>R.</given-names></name><name><surname>Ara&#xFA;jo</surname><given-names>F. V.</given-names></name><name><surname>Hajdu</surname><given-names>E.</given-names></name><name><surname>Peixoto</surname><given-names>R.</given-names></name><name><surname>Lins</surname><given-names>U.</given-names></name><etal/></person-group>. (<year>2009</year>). <article-title>Diversity of bacteria in the marine sponge <italic>Aplysina fulva</italic> in Brazilian coastal waters</article-title>. <source>Appl. Environ. Microbiol</source>. <volume>75</volume>, <fpage>3331</fpage>&#x2013;<lpage>3343</lpage>. <pub-id pub-id-type=\"doi\">10.1128/AEM.02101-08</pub-id><pub-id pub-id-type=\"pmid\">19304829</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B26\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hentschel</surname><given-names>U.</given-names></name><name><surname>Hopke</surname><given-names>J.</given-names></name><name><surname>Horn</surname><given-names>M.</given-names></name><name><surname>Friedrich</surname><given-names>A. B.</given-names></name><name><surname>Wagner</surname><given-names>M.</given-names></name><name><surname>Hacker</surname><given-names>J.</given-names></name><etal/></person-group>. (<year>2002</year>). <article-title>Molecular evidence for a uniform microbial community in sponges from different oceans</article-title>. <source>Appl. Environ. Microbiol</source>. <volume>68</volume>, <fpage>4431</fpage>&#x2013;<lpage>4440</lpage>. <pub-id pub-id-type=\"doi\">10.1128/AEM.68.9.4431-4440.2002</pub-id><pub-id pub-id-type=\"pmid\">12200297</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B27\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hentschel</surname><given-names>U.</given-names></name><name><surname>Usher</surname><given-names>K. M.</given-names></name><name><surname>Taylor</surname><given-names>M. W.</given-names></name></person-group> (<year>2006</year>). <article-title>Marine sponges as microbial fermenters</article-title>. <source>FEMS Microbiol. Ecol</source>. <volume>55</volume>, <fpage>167</fpage>&#x2013;<lpage>177</lpage>. <pub-id pub-id-type=\"doi\">10.1111/j.1574-6941.2005.00046.x</pub-id><pub-id pub-id-type=\"pmid\">16420625</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B28\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hoffmann</surname><given-names>F.</given-names></name><name><surname>Radax</surname><given-names>R.</given-names></name><name><surname>Woebken</surname><given-names>D.</given-names></name><name><surname>Holtappels</surname><given-names>M.</given-names></name><name><surname>Lavik</surname><given-names>G.</given-names></name><name><surname>Rapp</surname><given-names>H. T.</given-names></name><etal/></person-group>. (<year>2009</year>). <article-title>Complex nitrogen cycling in the sponge <italic>Geodia barretti</italic></article-title>. <source>Environ. Microbiol</source>. <volume>11</volume>, <fpage>2228</fpage>&#x2013;<lpage>2243</lpage>. <pub-id pub-id-type=\"doi\">10.1111/j.1462-2920.2009.01944.x</pub-id><pub-id pub-id-type=\"pmid\">19453700</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B29\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jackson</surname><given-names>S. A.</given-names></name><name><surname>Flemer</surname><given-names>B.</given-names></name><name><surname>McCann</surname><given-names>A.</given-names></name><name><surname>Kennedy</surname><given-names>J.</given-names></name><name><surname>Morrissey</surname><given-names>J. P.</given-names></name><name><surname>O'Gara</surname><given-names>F.</given-names></name><etal/></person-group>. (<year>2013</year>). <article-title>Archaea appear to dominate the microbiome of Inflatella pellicula deep sea sponges</article-title>. <source>PLoS ONE</source>\n<volume>8</volume>:<fpage>e084438</fpage>. <pub-id pub-id-type=\"doi\">10.1371/journal.pone.0084438</pub-id><pub-id pub-id-type=\"pmid\">24386380</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B30\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kamke</surname><given-names>J.</given-names></name><name><surname>Taylor</surname><given-names>M. W.</given-names></name><name><surname>Schmitt</surname><given-names>S.</given-names></name></person-group> (<year>2010</year>). <article-title>Activity profiles for marine sponge-associated bacteria obtained by 16S rRNA vs. 16S rRNA gene comparisons</article-title>. <source>ISME J</source>. <volume>4</volume>, <fpage>498</fpage>&#x2013;<lpage>508</lpage>. <pub-id pub-id-type=\"doi\">10.1038/ismej.2009.143</pub-id><pub-id pub-id-type=\"pmid\">20054355</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B31\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Klindworth</surname><given-names>A.</given-names></name><name><surname>Pruesse</surname><given-names>E.</given-names></name><name><surname>Schweer</surname><given-names>T.</given-names></name><name><surname>Peplies</surname><given-names>J.</given-names></name><name><surname>Quast</surname><given-names>C.</given-names></name><name><surname>Horn</surname><given-names>M.</given-names></name><etal/></person-group>. (<year>2013</year>). <article-title>Evaluation of general 16S ribosomal RNA gene PCR primers for classical and next-generation sequencing-based diversity studies</article-title>. <source>Nucleic Acids Res</source>. <volume>41</volume>, <fpage>1</fpage>&#x2013;<lpage>11</lpage>. <pub-id pub-id-type=\"doi\">10.1093/nar/gks808</pub-id><pub-id pub-id-type=\"pmid\">23143271</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B32\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>K&#xF6;nneke</surname><given-names>M.</given-names></name><name><surname>Bernhard</surname><given-names>A. E.</given-names></name><name><surname>de la Torre</surname><given-names>J. R.</given-names></name><name><surname>Walker</surname><given-names>C. B.</given-names></name><name><surname>Waterbury</surname><given-names>J. B.</given-names></name><name><surname>Stahl</surname><given-names>D. A.</given-names></name></person-group> (<year>2005</year>). <article-title>Isolation of an autotrophic ammonia-oxidizing marine archaeon</article-title>. <source>Nature</source>\n<volume>437</volume>, <fpage>543</fpage>&#x2013;<lpage>546</lpage>. <pub-id pub-id-type=\"doi\">10.1038/nature03911</pub-id><pub-id pub-id-type=\"pmid\">16177789</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B33\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kunin</surname><given-names>V.</given-names></name><name><surname>Engelbrektson</surname><given-names>A.</given-names></name><name><surname>Ochman</surname><given-names>H.</given-names></name><name><surname>Hugenholtz</surname><given-names>P.</given-names></name></person-group> (<year>2010</year>). <article-title>Wrinkles in the rare biosphere: pyrosequencing errors can lead to artificial inflation of diversity estimates</article-title>. <source>Environ. Microbiol</source>. <volume>12</volume>, <fpage>118</fpage>&#x2013;<lpage>123</lpage>. <pub-id pub-id-type=\"doi\">10.1111/j.1462-2920.2009.02051.x</pub-id><pub-id pub-id-type=\"pmid\">19725865</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B34\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>O. O.</given-names></name><name><surname>Chiu</surname><given-names>P. Y.</given-names></name><name><surname>Wong</surname><given-names>Y. H.</given-names></name><name><surname>Pawlik</surname><given-names>J. R.</given-names></name><name><surname>Qian</surname><given-names>P. Y.</given-names></name></person-group> (<year>2009</year>). <article-title>Evidence for vertical transmission of bacterial symbionts from adult to embryo in the caribbean sponge <italic>Svenzea zeai</italic></article-title>. <source>Appl. Environ. Microbiol</source>. <volume>75</volume>, <fpage>6147</fpage>&#x2013;<lpage>6156</lpage>. <pub-id pub-id-type=\"doi\">10.1128/AEM.00023-09</pub-id><pub-id pub-id-type=\"pmid\">19648378</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B35\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>Y. K.</given-names></name><name><surname>Lee</surname><given-names>J.</given-names></name><name><surname>Lee</surname><given-names>H. K.</given-names></name></person-group> (<year>2001</year>). <article-title>Microbial symbiosis in marine sponges</article-title>. <source>J. Microbiol</source>. <volume>39</volume>, <fpage>254</fpage>&#x2013;<lpage>264</lpage>.</mixed-citation>\n      </ref>\n      <ref id=\"B36\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>Z. Y.</given-names></name><name><surname>Wang</surname><given-names>Y. Z.</given-names></name><name><surname>He</surname><given-names>L. M.</given-names></name><name><surname>Zheng</surname><given-names>H. J.</given-names></name></person-group> (<year>2014</year>). <article-title>Metabolic profiles of prokaryotic and eukaryotic communities in deep-sea sponge Lamellomorpha sp. indicated by metagenomics</article-title>. <source>Sci. Rep</source>. <volume>4</volume>, <fpage>1</fpage>&#x2013;<lpage>11</lpage>. <pub-id pub-id-type=\"doi\">10.1038/srep03895</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B37\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>K. L.</given-names></name><name><surname>Porras-Alfaro</surname><given-names>A.</given-names></name><name><surname>Kuske</surname><given-names>C. R.</given-names></name><name><surname>Eichorst</surname><given-names>S. A.</given-names></name><name><surname>Xie</surname><given-names>G.</given-names></name></person-group> (<year>2012</year>). <article-title>Accurate, rapid taxonomic classification of fungal large-subunit rRNA genes</article-title>. <source>Appl. Environ. Microbiol</source>. <volume>78</volume>, <fpage>1523</fpage>&#x2013;<lpage>1533</lpage>. <pub-id pub-id-type=\"doi\">10.1128/AEM.06826-11</pub-id><pub-id pub-id-type=\"pmid\">22194300</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B38\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Maldonado</surname><given-names>M.</given-names></name><name><surname>Cortadellas</surname><given-names>N.</given-names></name><name><surname>Trillas</surname><given-names>M. I.</given-names></name><name><surname>R&#xFC;tzler</surname><given-names>K.</given-names></name></person-group> (<year>2005</year>). <article-title>Endosymbiotic yeast maternally transmitted in a marine sponge</article-title>. <source>Biol. Bull</source>. <volume>209</volume>, <fpage>94</fpage>&#x2013;<lpage>106</lpage>. <pub-id pub-id-type=\"doi\">10.2307/3593127</pub-id><pub-id pub-id-type=\"pmid\">16260769</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B39\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Naim</surname><given-names>M. A.</given-names></name><name><surname>Smidt</surname><given-names>H.</given-names></name><name><surname>Sipkema</surname><given-names>D.</given-names></name></person-group> (<year>2017</year>). <article-title>Fungi found in Mediterranean and North Sea sponges: how specific are they?</article-title>\n<source>PeerJ</source>\n<volume>5</volume>:<fpage>e3722</fpage>. <pub-id pub-id-type=\"doi\">10.7717/peerj.3722</pub-id><pub-id pub-id-type=\"pmid\">28894639</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B40\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Nilsson</surname><given-names>R. H.</given-names></name><name><surname>Veldre</surname><given-names>V.</given-names></name><name><surname>Hartmann</surname><given-names>M.</given-names></name><name><surname>Unterseher</surname><given-names>M.</given-names></name><name><surname>Amend</surname><given-names>A.</given-names></name><name><surname>Bergsten</surname><given-names>J.</given-names></name><etal/></person-group> (<year>2010</year>). <article-title>An open source software package for automated extraction of ITS1 and ITS2 from fungal ITS sequences for use in high-throughput community assays and molecular ecology</article-title>. <source>Fungal Ecol</source>. <volume>3</volume>, <fpage>284</fpage>&#x2013;<lpage>287</lpage>. <pub-id pub-id-type=\"doi\">10.1016/j.funeco.2010.05.002</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B41\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>&#xD6;zt&#xFC;rk</surname><given-names>B.</given-names></name><name><surname>de Jaeger</surname><given-names>L.</given-names></name><name><surname>Smidt</surname><given-names>H.</given-names></name><name><surname>Sipkema</surname><given-names>D.</given-names></name></person-group> (<year>2013</year>). <article-title>Culture-dependent and independent approaches for identifying novel halogenases encoded by <italic>Crambe crambe</italic> (marine sponge) microbiota</article-title>. <source>Sci. Rep</source>. <volume>3</volume>:<fpage>2780</fpage>. <pub-id pub-id-type=\"doi\">10.1038/srep02780</pub-id><pub-id pub-id-type=\"pmid\">24071658</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B42\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Pape</surname><given-names>T.</given-names></name><name><surname>Hoffmann</surname><given-names>F.</given-names></name><name><surname>Qu&#xE9;ric</surname><given-names>N.-V.</given-names></name><name><surname>von Juterzenka</surname><given-names>K.</given-names></name><name><surname>Reitner</surname><given-names>J.</given-names></name><name><surname>Michaelis</surname><given-names>W.</given-names></name></person-group> (<year>2006</year>). <article-title>Dense populations of Archaea associated with the demosponge <italic>Tentorium semisuberites</italic> Schmidt, 1870 from Arctic deep-waters</article-title>. <source>Polar Biol</source>. <volume>29</volume>, <fpage>662</fpage>&#x2013;<lpage>667</lpage>. <pub-id pub-id-type=\"doi\">10.1007/s00300-005-0103-4</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B43\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Pruesse</surname><given-names>E.</given-names></name><name><surname>Quast</surname><given-names>C.</given-names></name><name><surname>Knittel</surname><given-names>K.</given-names></name><name><surname>Fuchs</surname><given-names>B. M.</given-names></name><name><surname>Ludwig</surname><given-names>W.</given-names></name><name><surname>Peplies</surname><given-names>J.</given-names></name><etal/></person-group>. (<year>2007</year>). <article-title>SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB</article-title>. <source>Nucleic Acids Res</source>. <volume>35</volume>, <fpage>7188</fpage>&#x2013;<lpage>7196</lpage>. <pub-id pub-id-type=\"doi\">10.1093/nar/gkm864</pub-id><pub-id pub-id-type=\"pmid\">17947321</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B44\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>R&#xE4;m&#xE4;</surname><given-names>T.</given-names></name><name><surname>Nord&#xE9;n</surname><given-names>J.</given-names></name><name><surname>Davey</surname><given-names>M. L.</given-names></name><name><surname>Mathiassen</surname><given-names>G. H.</given-names></name><name><surname>Spatafora</surname><given-names>J. W.</given-names></name><name><surname>Kauserud</surname><given-names>H.</given-names></name></person-group> (<year>2014</year>). <article-title>Fungi ahoy! diversity on marine wooden substrata in the high North</article-title>. <source>Fungal Ecol</source>. <volume>8</volume>, <fpage>46</fpage>&#x2013;<lpage>58</lpage>. <pub-id pub-id-type=\"doi\">10.1016/j.funeco.2013.12.002</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B45\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rodr&#xED;guez-Marconi</surname><given-names>S.</given-names></name><name><surname>De La Iglesia</surname><given-names>R.</given-names></name><name><surname>D&#xED;ez</surname><given-names>B.</given-names></name><name><surname>Fonseca</surname><given-names>C. A.</given-names></name><name><surname>Hajdu</surname><given-names>E.</given-names></name><name><surname>Trefault</surname><given-names>N.</given-names></name><etal/></person-group>. (<year>2015</year>). <article-title>Characterization of bacterial, archaeal and eukaryote symbionts from Antarctic sponges reveals a high diversity at a three-domain level and a particular signature for this ecosystem</article-title>. <source>PLoS ONE</source>\n<volume>10</volume>:<fpage>e138837</fpage>. <pub-id pub-id-type=\"doi\">10.1371/journal.pone.0138837</pub-id><pub-id pub-id-type=\"pmid\">26421612</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B46\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Schmitt</surname><given-names>S.</given-names></name><name><surname>Hentschel</surname><given-names>U.</given-names></name><name><surname>Taylor</surname><given-names>M. W.</given-names></name></person-group> (<year>2011</year>). <article-title>Deep sequencing reveals diversity and community structure of complex microbiota in five Mediterranean sponges</article-title>. <source>Hydrobiologia</source>\n<volume>687</volume>, <fpage>341</fpage>&#x2013;<lpage>351</lpage>. <pub-id pub-id-type=\"doi\">10.1007/s10750-011-0799-9</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B47\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Schmitt</surname><given-names>S.</given-names></name><name><surname>Tsai</surname><given-names>P.</given-names></name><name><surname>Bell</surname><given-names>J.</given-names></name><name><surname>Fromont</surname><given-names>J.</given-names></name><name><surname>Ilan</surname><given-names>M.</given-names></name><name><surname>Lindquist</surname><given-names>N.</given-names></name><etal/></person-group>. (<year>2012</year>). <article-title>Assessing the complex sponge microbiota: core, variable and species-specific bacterial communities in marine sponges</article-title>. <source>ISME J</source>. <volume>6</volume>, <fpage>564</fpage>&#x2013;<lpage>576</lpage>. <pub-id pub-id-type=\"doi\">10.1038/ismej.2011.116</pub-id><pub-id pub-id-type=\"pmid\">21993395</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B48\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sharp</surname><given-names>K. H.</given-names></name><name><surname>Eam</surname><given-names>B.</given-names></name><name><surname>John Faulkner</surname><given-names>D. J.</given-names></name><name><surname>Haygood</surname><given-names>M. G.</given-names></name></person-group> (<year>2007</year>). <article-title>Vertical transmission of diverse microbes in the tropical sponge Corticium sp</article-title>. <source>Appl. Environ. Microbiol</source>. <volume>73</volume>, <fpage>622</fpage>&#x2013;<lpage>629</lpage>. <pub-id pub-id-type=\"doi\">10.1128/AEM.01493-06</pub-id><pub-id pub-id-type=\"pmid\">17122394</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B49\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Simister</surname><given-names>R. L.</given-names></name><name><surname>Deines</surname><given-names>P.</given-names></name><name><surname>Bott&#xE9;</surname><given-names>E. S.</given-names></name><name><surname>Webster</surname><given-names>N. S.</given-names></name><name><surname>Taylor</surname><given-names>M. W.</given-names></name></person-group> (<year>2012</year>). <article-title>Sponge-specific clusters revisited: a comprehensive phylogeny of sponge-associated microorganisms</article-title>. <source>Environ. Microbiol</source>. <volume>14</volume>, <fpage>517</fpage>&#x2013;<lpage>524</lpage>. <pub-id pub-id-type=\"doi\">10.1111/j.1462-2920.2011.02664.x</pub-id><pub-id pub-id-type=\"pmid\">22151434</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B50\">\n        <mixed-citation publication-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Simpson</surname><given-names>T. L.</given-names></name></person-group> (<year>1984</year>). <source>The Cell Biology of Sponges.</source>\n<publisher-loc>New York, NY</publisher-loc>: <publisher-name>Springer- verlag</publisher-name>.</mixed-citation>\n      </ref>\n      <ref id=\"B51\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sipkema</surname><given-names>D.</given-names></name><name><surname>Blanch</surname><given-names>H. W.</given-names></name></person-group> (<year>2009</year>). <article-title>Spatial distribution of bacteria associated with the marine sponge <italic>Tethya californiana</italic></article-title>. <source>Mar. Biol</source>. <volume>157</volume>, <fpage>627</fpage>&#x2013;<lpage>638</lpage>. <pub-id pub-id-type=\"doi\">10.1007/s00227-009-1347-2</pub-id><pub-id pub-id-type=\"pmid\">24391242</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B52\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sipkema</surname><given-names>D.</given-names></name><name><surname>de Caralt</surname><given-names>S.</given-names></name><name><surname>Morillo</surname><given-names>J. A.</given-names></name><name><surname>Al-Soud</surname><given-names>W. A.</given-names></name><name><surname>S&#xF8;rensen</surname><given-names>S. J.</given-names></name><name><surname>Smidt</surname><given-names>H.</given-names></name><etal/></person-group>. (<year>2015</year>). <article-title>Similar sponge-associated bacteria can be acquired via both vertical and horizontal transmission</article-title>. <source>Environ. Microbiol.</source>\n<volume>17</volume>, <fpage>3807</fpage>&#x2013;<lpage>3821</lpage>. <pub-id pub-id-type=\"doi\">10.1111/1462-2920.12827</pub-id><pub-id pub-id-type=\"pmid\">25732544</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B53\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Steger</surname><given-names>D.</given-names></name><name><surname>Ettinger-Epstein</surname><given-names>P.</given-names></name><name><surname>Whalan</surname><given-names>S.</given-names></name><name><surname>Hentschel</surname><given-names>U.</given-names></name><name><surname>de Nys</surname><given-names>R.</given-names></name><name><surname>Wagner</surname><given-names>M.</given-names></name><etal/></person-group>. (<year>2008</year>). <article-title>Diversity and mode of transmission of ammonia-oxidizing archaea in marine sponges</article-title>. <source>Environ. Microbiol</source>. <volume>10</volume>, <fpage>1087</fpage>&#x2013;<lpage>1094</lpage>. <pub-id pub-id-type=\"doi\">10.1111/j.1462-2920.2007.01515.x</pub-id><pub-id pub-id-type=\"pmid\">18177367</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B54\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Taylor</surname><given-names>M. W.</given-names></name><name><surname>Radax</surname><given-names>R.</given-names></name><name><surname>Steger</surname><given-names>D.</given-names></name><name><surname>Wagner</surname><given-names>M.</given-names></name></person-group> (<year>2007</year>). <article-title>Sponge-associated microorganisms: evolution, ecology, and biotechnological potential</article-title>. <source>Microbiol. Mol. Biol. Rev</source>. <volume>71</volume>, <fpage>295</fpage>&#x2013;<lpage>347</lpage>. <pub-id pub-id-type=\"doi\">10.1128/MMBR.00040-06</pub-id><pub-id pub-id-type=\"pmid\">17554047</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B55\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Taylor</surname><given-names>M. W.</given-names></name><name><surname>Tsai</surname><given-names>P.</given-names></name><name><surname>Simister</surname><given-names>R. L.</given-names></name><name><surname>Deines</surname><given-names>P.</given-names></name><name><surname>Botte</surname><given-names>E.</given-names></name><name><surname>Ericson</surname><given-names>G.</given-names></name><etal/></person-group>. (<year>2013</year>). <article-title>&#x201C;Sponge-specific&#x201D; bacteria are widespread (but rare) in diverse marine environments</article-title>. <source>ISME J</source>. <volume>7</volume>, <fpage>438</fpage>&#x2013;<lpage>443</lpage>. <pub-id pub-id-type=\"doi\">10.1038/ismej.2012.111</pub-id><pub-id pub-id-type=\"pmid\">23038173</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B56\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Thomas</surname><given-names>T.</given-names></name><name><surname>Moitinho-Silva</surname><given-names>L.</given-names></name><name><surname>Lurgi</surname><given-names>M.</given-names></name><name><surname>Bj&#xF6;rk</surname><given-names>J. R.</given-names></name><name><surname>Easson</surname><given-names>C.</given-names></name><name><surname>Astudillo-Garc&#xED;a</surname><given-names>C.</given-names></name><etal/></person-group>. (<year>2016</year>). <article-title>Diversity, structure and convergent evolution of the global sponge microbiome</article-title>. <source>Nat. Commun</source>. <volume>7</volume>:<fpage>11870</fpage>. <pub-id pub-id-type=\"doi\">10.1038/ncomms11870</pub-id><pub-id pub-id-type=\"pmid\">27306690</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B57\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Turque</surname><given-names>A. S.</given-names></name><name><surname>Batista</surname><given-names>D.</given-names></name><name><surname>Silveira</surname><given-names>C. B.</given-names></name><name><surname>Cardoso</surname><given-names>A. M.</given-names></name><name><surname>Vieira</surname><given-names>R. P.</given-names></name><name><surname>Moraes</surname><given-names>F. C.</given-names></name><etal/></person-group>. (<year>2010</year>). <article-title>Environmental shaping of sponge associated archaeal communities</article-title>. <source>PLoS ONE</source>\n<volume>5</volume>:<fpage>e15774</fpage>. <pub-id pub-id-type=\"doi\">10.1371/journal.pone.0015774</pub-id><pub-id pub-id-type=\"pmid\">21209889</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B58\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Usher</surname><given-names>K. M.</given-names></name></person-group> (<year>2008</year>). <article-title>The ecology and phylogeny of cyanobacterial symbionts in sponges</article-title>. <source>Mar. Ecol</source>. <volume>29</volume>, <fpage>178</fpage>&#x2013;<lpage>192</lpage>. <pub-id pub-id-type=\"doi\">10.1111/j.1439-0485.2008.00245.x</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B59\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Usher</surname><given-names>K. M.</given-names></name><name><surname>Bergman</surname><given-names>B.</given-names></name><name><surname>Raven</surname><given-names>J. a</given-names></name></person-group> (<year>2007</year>). <article-title>Exploring cyanobacterial mutualisms</article-title>. <source>Annu. Rev. Ecol. Evol. Syst</source>. <volume>38</volume>, <fpage>255</fpage>&#x2013;<lpage>273</lpage>. <pub-id pub-id-type=\"doi\">10.1146/annurev.ecolsys.38.091206.095641</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B60\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Usher</surname><given-names>K. M.</given-names></name><name><surname>Fromont</surname><given-names>J.</given-names></name><name><surname>Sutton</surname><given-names>D. C.</given-names></name><name><surname>Toze</surname><given-names>S.</given-names></name></person-group> (<year>2004</year>). <article-title>The biogeography and phylogeny of unicellular cyanobacterial symbionts in sponges from Australia and the Mediterranean</article-title>. <source>Microb. Ecol</source>. <volume>48</volume>, <fpage>167</fpage>&#x2013;<lpage>177</lpage>. <pub-id pub-id-type=\"doi\">10.1007/s00248-003-1062-3</pub-id><pub-id pub-id-type=\"pmid\">15546037</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B61\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vacelet</surname><given-names>J.</given-names></name><name><surname>Donadey</surname><given-names>C.</given-names></name></person-group> (<year>1977</year>). <article-title>Electron microscope study of the association between some sponges and bacteria</article-title>. <source>J. Exp. Biol. Ecol</source>. <volume>30</volume>, <fpage>301</fpage>&#x2013;<lpage>314</lpage>. <pub-id pub-id-type=\"doi\">10.1016/0022-0981(77)90038-7</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B62\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vogel</surname><given-names>S.</given-names></name></person-group> (<year>1977</year>). <article-title>Current-induced flow through living sponges in nature</article-title>. <source>Proc. Natl. Acad. Sci. U.S.A</source>. <volume>74</volume>, <fpage>2069</fpage>&#x2013;<lpage>2071</lpage>. <pub-id pub-id-type=\"doi\">10.1073/pnas.74.5.2069</pub-id><pub-id pub-id-type=\"pmid\">266728</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B63\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Webster</surname><given-names>N. S.</given-names></name><name><surname>Blackall</surname><given-names>L. L.</given-names></name></person-group> (<year>2009</year>). <article-title>What do we really know about sponge-microbial symbioses?</article-title>\n<source>ISME J</source>. <volume>3</volume>, <fpage>1</fpage>&#x2013;<lpage>3</lpage>. <pub-id pub-id-type=\"doi\">10.1038/ismej.2008.102</pub-id><pub-id pub-id-type=\"pmid\">18971962</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B64\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Webster</surname><given-names>N. S.</given-names></name><name><surname>Taylor</surname><given-names>M. W.</given-names></name></person-group> (<year>2012</year>). <article-title>Marine sponges and their microbial symbionts: love and other relationships</article-title>. <source>Environ. Microbiol</source>. <volume>14</volume>, <fpage>335</fpage>&#x2013;<lpage>346</lpage>. <pub-id pub-id-type=\"doi\">10.1111/j.1462-2920.2011.02460.x</pub-id><pub-id pub-id-type=\"pmid\">21443739</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B65\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Webster</surname><given-names>N. S.</given-names></name><name><surname>Negri</surname><given-names>A. P.</given-names></name><name><surname>Munro</surname><given-names>M. M.</given-names></name><name><surname>Battershill</surname><given-names>C. N.</given-names></name></person-group> (<year>2004</year>). <article-title>Diverse microbial communities inhabit Antarctic sponges</article-title>. <source>Environ. Microbiol</source>. <volume>6</volume>, <fpage>288</fpage>&#x2013;<lpage>300</lpage>. <pub-id pub-id-type=\"doi\">10.1111/j.1462-2920.2004.00570.x</pub-id><pub-id pub-id-type=\"pmid\">14871212</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B66\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Webster</surname><given-names>N. S.</given-names></name><name><surname>Taylor</surname><given-names>M. W.</given-names></name><name><surname>Behnam</surname><given-names>F.</given-names></name><name><surname>L&#xFC;cker</surname><given-names>S.</given-names></name><name><surname>Rattei</surname><given-names>T.</given-names></name><name><surname>Whalan</surname><given-names>S.</given-names></name><etal/></person-group>. (<year>2010</year>). <article-title>Deep sequencing reveals exceptional diversity and modes of transmission for bacterial sponge symbionts</article-title>. <source>Environ. Microbiol</source>. <volume>12</volume>, <fpage>2070</fpage>&#x2013;<lpage>2082</lpage>. <pub-id pub-id-type=\"doi\">10.1111/j.1462-2920.2009.02065.x</pub-id><pub-id pub-id-type=\"pmid\">21966903</pub-id></mixed-citation>\n      </ref>\n      <ref id=\"B67\">\n        <mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wilms</surname><given-names>R.</given-names></name><name><surname>Sass</surname><given-names>H.</given-names></name><name><surname>Koepke</surname><given-names>B.</given-names></name><name><surname>Cypionka</surname><given-names>H.</given-names></name><name><surname>Engelen</surname><given-names>B.</given-names></name></person-group> (<year>2006</year>). <article-title>Specific bacterial, archeal, and eukaryotic communities in tidal flat sediments along a vertical profile of several meters</article-title>. <source>Appl. Env. Microbiol.</source>\n<volume>72</volume>, <fpage>2756</fpage>&#x2013;<lpage>2764</lpage>. <pub-id pub-id-type=\"doi\">10.1128/AEM.72.4.2756-2764.2006</pub-id><pub-id pub-id-type=\"pmid\">16597980</pub-id></mixed-citation>\n      </ref>\n    </ref-list>\n  </back>\n</article>\n</pmc-articleset>\n"
     ]
    }
   ],
   "source": [
    "# print record\n",
    "# print list(root[0][0][1][17])\n",
    "print 'TITLE: ', root[0][0][1][4][0].text\n",
    "print 'ABSTRACT: ', root[0][0][1][17][0].text\n",
    "print record"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    ""
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2.0
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}